# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# greywalk <workisgreat@yandex.com>, 2012
msgid ""
msgstr ""
"Project-Id-Version: Sabayon Entropy Client\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-05-01 23:51+0200\n"
"PO-Revision-Date: 2013-05-01 21:51+0000\n"
"Last-Translator: lxnay <lxnay@sabayon.org>\n"
"Language-Team: Russian (http://www.transifex.com/projects/p/sabayon-entropy/language/ru/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ru\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: ../../lib/entropy/db/sqlite.py:1685
msgid "Exporting database table"
msgstr "Экспорт таблицы базы данных"

#: ../../lib/entropy/db/sqlite.py:1725
msgid "Database Export complete."
msgstr "Экспорт базы данных завершен."

#: ../../lib/entropy/db/sqlite.py:2103 ../../lib/entropy/db/sqlite.py:2207
#: ../../lib/entropy/db/sqlite.py:2315 ../../lib/entropy/db/sqlite.py:2328
#: ../../lib/entropy/db/sql.py:5437 ../../lib/entropy/db/sql.py:5450
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2610
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2748
#: ../../lib/entropy/client/interfaces/client.py:950
msgid "ATTENTION"
msgstr "ВНИМАНИЕ"

#: ../../lib/entropy/db/sqlite.py:2105 ../../lib/entropy/db/sqlite.py:2209
msgid "updating repository metadata layout, please wait!"
msgstr ""

#: ../../lib/entropy/db/sqlite.py:2316
msgid "generating provided_libs metadata, please wait!"
msgstr "генерируются метаданные provided_libs, пожалуйста, подождите!"

#: ../../lib/entropy/db/sql.py:5438
msgid "Spm error occured"
msgstr ""

#: ../../lib/entropy/db/sql.py:5451
msgid "Spm Unique Identifier not found for"
msgstr ""

#: ../../lib/entropy/db/skel.py:1662
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:945
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:971
msgid "SPM"
msgstr "SPM"

#: ../../lib/entropy/db/skel.py:1663
msgid "Running packages metadata update"
msgstr "Ведется обновление метаданных пакетов"

#: ../../lib/entropy/db/skel.py:1664
msgid "it could take a while"
msgstr "это может занять время"

#: ../../lib/entropy/db/skel.py:1683 ../../lib/entropy/db/skel.py:1708
#: ../../lib/entropy/db/skel.py:1719
msgid "Entropy"
msgstr ""

#: ../../lib/entropy/db/skel.py:1684
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:946
msgid "action"
msgstr "действие"

#: ../../lib/entropy/db/skel.py:1709
msgid "package move actions complete"
msgstr "перемещение пакета завершено"

#: ../../lib/entropy/db/skel.py:1720
msgid "package moves completed successfully"
msgstr "перемещения пакета успешно завершены"

#: ../../lib/entropy/db/skel.py:3730
msgid "Syncing current database"
msgstr "Синхронизация текущей базы данных"

#: ../../lib/entropy/db/skel.py:3731
#: ../../lib/entropy/client/interfaces/db.py:300
msgid "please wait"
msgstr "пожалуйста, подождите"

#: ../../lib/entropy/db/skel.py:3746
msgid "Removing entry"
msgstr "Удаление записи"

#: ../../lib/entropy/db/skel.py:3765
msgid "Adding entry"
msgstr "Добавление записи"

#: ../../lib/entropy/core/settings/base.py:2039
msgid "No description"
msgstr ""

#: ../../lib/kswitch/__init__.py:119
msgid "has been installed."
msgstr ""

#: ../../lib/kswitch/__init__.py:120
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3385
#: ../../lib/entropy/client/interfaces/db.py:438
#: ../../lib/entropy/client/interfaces/db.py:625
#: ../../lib/entropy/client/interfaces/db.py:1986
#: ../../lib/entropy/client/interfaces/db.py:1999 ../../lib/entropy/qa.py:842
#: ../../client/solo/commands/install.py:488
#: ../../client/solo/commands/install.py:491
#: ../../client/solo/commands/install.py:498
#: ../../client/solo/commands/install.py:501
#: ../../client/solo/commands/rescue.py:307
#: ../../client/solo/commands/rescue.py:318
msgid "Attention"
msgstr "Внимание"

#: ../../lib/kswitch/__init__.py:121
msgid "some external drivers cannot work across multiple kernels."
msgstr ""

#: ../../lib/kswitch/__init__.py:122
msgid "Please reboot your computer now !"
msgstr ""

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:247
msgid "Accessibility"
msgstr "Специальные возможности"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:249
msgid "Accessibility applications"
msgstr "Приложения специальных возможностей"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:253
msgid "Office"
msgstr "Офис"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:254
msgid "Applications used in office environments"
msgstr "Приложения, используемые в офисной среде"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:260
msgid "Development"
msgstr "Разработка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:261
msgid "Applications or system libraries"
msgstr "Приложения или системные библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:265
msgid "System"
msgstr "Система"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:266
msgid "System applications or libraries"
msgstr "Системные приложения или библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:270
msgid "Games"
msgstr "Игры"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:271
msgid "Games, enjoy your spare time"
msgstr "Игры, наслаждайтесь своим отдыхом"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:275
msgid "GNOME Desktop"
msgstr "Среда GNOME"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:277
msgid "Applications and libraries for the GNOME Desktop"
msgstr "Приложения и библиотеки для среды GNOME"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:281
msgid "KDE Desktop"
msgstr "Среда KDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:283
msgid "Applications and libraries for the KDE Desktop"
msgstr "Приложения и библиотеки для среды KDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:287
msgid "XFCE Desktop"
msgstr "Среда XFCE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:289
msgid "Applications and libraries for the XFCE Desktop"
msgstr "Приложения и библиотеки для среды XFCE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:293
msgid "LXDE Desktop"
msgstr "Среда LXDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:295
msgid "Applications and libraries for the LXDE Desktop"
msgstr "Приложения и библиотеки для среды LXDE"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:299
msgid "Multimedia"
msgstr "Мультимедиа"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:301
msgid "Applications and libraries for Multimedia"
msgstr "Приложения и библиотеки для мультимедиа"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:305
msgid "Networking"
msgstr "Сеть"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:307
msgid "Applications and libraries for Networking"
msgstr "Приложения и библиотеки для подключения к сети"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:311
msgid "Science"
msgstr "Наука"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:313
msgid "Scientific applications and libraries"
msgstr "Научные приложения и библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:317
msgid "Security"
msgstr "Безопасность"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:319
msgid "Security oriented applications"
msgstr "Приложения раздела \"Безопасность\""

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:323
msgid "X11"
msgstr "X11"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:325
msgid "Applications and libraries for X11"
msgstr "Приложения и библиотеки для X11"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:609
msgid "Reloading Portage modules"
msgstr "Модули Portage перезагружаются"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:972
msgid "updating metadata"
msgstr ""

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:1139
msgid "error"
msgstr "ошибка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:1141
#: ../../lib/entropy/client/interfaces/package.py:692
#: ../../lib/entropy/client/interfaces/package.py:1373
msgid "not found"
msgstr "не найдено"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2569
msgid "package not available on system"
msgstr "пакет недоступен в системе"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2578
msgid "repackaging"
msgstr "перепаковка"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2590
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2785
msgid "WARNING"
msgstr "ОСТОРОЖНО"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2591
msgid "Cannot complete quickpkg for atom"
msgstr "Невозможно завершить quickpkg для атома"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2593
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2788
msgid "do it manually"
msgstr "сделайте это вручную"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2611
msgid "package files rebuild did not run properly"
msgstr "пересборка файлов пакета не запущена корректно"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2612
msgid "Please update packages manually"
msgstr "Будьте добры обновить пакеты вручную"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2749
msgid "forcing package updates"
msgstr "принуждение к обновлению пакетов"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2750
msgid "Syncing with"
msgstr "Синхронизация с"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:2786
msgid "Cannot complete quickpkg for atoms"
msgstr "Невозможно завершить quickpkg для атомов"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3142
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4824
#: ../../lib/entropy/client/interfaces/trigger.py:286
#: ../../lib/entropy/client/interfaces/trigger.py:549
msgid "QA"
msgstr "QA"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3143
msgid "Cannot update Portage package metadata"
msgstr ""

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3169
msgid "SPM uid update error"
msgstr "Ошибка при обновлении SPM uid"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3238
msgid "Cannot update SPM installed pkgs file"
msgstr "Невозможно обновить файл установленных пакетов SPM"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3331
msgid "Portage world file is corrupted"
msgstr "Файл Portage world поврежден"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3386
msgid "preserved libraries have been found on system"
msgstr "в системе найдены устаревшие библиотеки"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3527
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3677
msgid "does not exist"
msgstr "не существует"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3527
msgid "Overwriting"
msgstr "Перезапись"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3573
msgid "variable differs"
msgstr "всевозможные различия"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3573
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3662
msgid "Updating"
msgstr "Обновление"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3588
msgid "updating critical variables"
msgstr "Обновление критических переменных"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3662
msgid "differs"
msgstr "различия"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3677
msgid "Reverting"
msgstr "Возвращение"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3692
msgid "Skipping configuration files update, you are not root."
msgstr "Пропуск обновления файлов конфигурации, Вы не root."

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:3952
msgid "Error calculating dependencies"
msgstr "Ошибка при подсчёте зависимостей"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4511
msgid "not supported"
msgstr "не поддерживается"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4512
msgid "Probably Portage API has changed"
msgstr "Возможно, был изменен API портеджа"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4575
#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4668
msgid "IOError while reading"
msgstr "Ошибка Ввода-Вывода при чтении"

#: ../../lib/entropy/spm/plugins/interfaces/portage_plugin/__init__.py:4825
msgid "illegal Entropy package tag in ebuild"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:60
msgid "Unknown documents"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:61
msgid "Comments"
msgstr "Комментарии"

#. BBCODE_TYPE_ID: _('BBcode Documents'),
#: ../../lib/entropy/client/services/interfaces.py:63
msgid "Images/Screenshots"
msgstr "Изображения/скриншоты"

#: ../../lib/entropy/client/services/interfaces.py:64
msgid "Generic Files"
msgstr "Общие файлы"

#: ../../lib/entropy/client/services/interfaces.py:65
msgid "Videos"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:66
msgid "Icons"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:69
msgid "Unknown document"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:70
msgid "Comment"
msgstr "Комментарий"

#. BBCODE_TYPE_ID: _('BBcode Document'),
#: ../../lib/entropy/client/services/interfaces.py:72
msgid "Image/Screenshot"
msgstr "Изображение/скриншот"

#: ../../lib/entropy/client/services/interfaces.py:73
msgid "Generic File"
msgstr "Общий файл"

#: ../../lib/entropy/client/services/interfaces.py:74
msgid "Video"
msgstr ""

#: ../../lib/entropy/client/services/interfaces.py:75
msgid "Icon"
msgstr ""

#: ../../lib/entropy/client/misc.py:213 ../../lib/entropy/client/misc.py:579
msgid "Automerging file"
msgstr "Автообъединение файла"

#: ../../lib/entropy/client/misc.py:247
msgid "Found update"
msgstr ""

#: ../../lib/entropy/client/misc.py:595
msgid "System Error"
msgstr "Системная ошибка"

#: ../../lib/entropy/client/misc.py:596
msgid "Cannot automerge file"
msgstr "Невозможно автоматически объединить файл"

#: ../../lib/entropy/client/interfaces/client.py:951
msgid "forcing packages metadata update"
msgstr "принуждение к обновлению метаинформации пакетов"

#: ../../lib/entropy/client/interfaces/client.py:960
msgid "Updating system database using repository"
msgstr "Обновление системной базы данных, используя ID репозитория"

#: ../../lib/entropy/client/interfaces/cache.py:106
#: ../../lib/entropy/client/interfaces/methods.py:100
#: ../../lib/entropy/client/interfaces/methods.py:125
#: ../../client/solo/commands/upgrade.py:297
msgid "Repository"
msgstr "Репозиторий"

#: ../../lib/entropy/client/interfaces/cache.py:107
#: ../../lib/entropy/client/interfaces/methods.py:126
msgid "is corrupted"
msgstr "испорчен"

#: ../../lib/entropy/client/interfaces/cache.py:108
msgid "Cannot calculate the checksum"
msgstr ""

#: ../../lib/entropy/client/interfaces/trigger.py:72
msgid "Source Package Manager interface can't be loaded"
msgstr "Пакетный менеджер источника не может быть загружен"

#: ../../lib/entropy/client/interfaces/trigger.py:73
#: ../../lib/entropy/client/interfaces/package.py:3532
#: ../../lib/entropy/client/interfaces/package.py:3569
#: ../../lib/entropy/client/interfaces/package.py:3696
#: ../../lib/entropy/client/interfaces/package.py:3703
#: ../../lib/entropy/client/interfaces/package.py:3759
#: ../../lib/entropy/client/interfaces/package.py:3787
#: ../../lib/entropy/client/interfaces/package.py:4035
#: ../../lib/entropy/client/interfaces/package.py:4055
#: ../../lib/entropy/security.py:658 ../../client/solo/commands/rescue.py:200
msgid "Error"
msgstr "Ошибка"

#: ../../lib/entropy/client/interfaces/trigger.py:75
msgid "please fix"
msgstr "пожалуйста, пофиксите"

#: ../../lib/entropy/client/interfaces/trigger.py:287
msgid "Cannot run External trigger for"
msgstr "Невозможно запустить Внешний триггер для"

#: ../../lib/entropy/client/interfaces/trigger.py:289
#: ../../lib/entropy/client/interfaces/trigger.py:553
msgid "Please report it"
msgstr "Пожалуйста, сообщите об этом"

#: ../../lib/entropy/client/interfaces/trigger.py:510
msgid "Installing info"
msgstr ""

#: ../../lib/entropy/client/interfaces/trigger.py:538
msgid "Package phase"
msgstr ""

#: ../../lib/entropy/client/interfaces/trigger.py:550
msgid "Cannot run phase"
msgstr ""

#: ../../lib/entropy/client/interfaces/trigger.py:564
#: ../../lib/entropy/client/interfaces/package.py:1836
msgid "Source Package Manager is too old, please update it"
msgstr ""

#: ../../lib/entropy/client/interfaces/trigger.py:577
msgid "Source Package Manager phase error"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:81
msgid "Please update your repositories now in order to remove this message!"
msgstr "Пожалуйста, обновите ваши репозитории сейчас, чтобы убрать это сообщение!"

#: ../../lib/entropy/client/interfaces/methods.py:101
#: ../../client/solo/commands/pkg.py:199
#: ../../client/solo/commands/mask.py:101
msgid "is not available"
msgstr "недоступен"

#: ../../lib/entropy/client/interfaces/methods.py:101
#: ../../lib/entropy/client/interfaces/methods.py:126
msgid "Cannot validate"
msgstr "Невозможно подтвердить"

#: ../../lib/entropy/client/interfaces/methods.py:274
msgid "bad repository id specified"
msgstr "выбран неправильный ID репозитория"

#: ../../lib/entropy/client/interfaces/methods.py:299
#, python-format
msgid "Repository %s hasn't been downloaded yet."
msgstr "Репозиторий %s ещё не скачан."

#: ../../lib/entropy/client/interfaces/methods.py:995
msgid "System database not found or corrupted"
msgstr "База данных системы не найдена или испорчена"

#: ../../lib/entropy/client/interfaces/methods.py:996
msgid "running in safe mode using temporary, empty repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1178
msgid "Backing up repository to"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1198
msgid "Repository backed up successfully"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1207
#: ../../lib/entropy/client/interfaces/methods.py:1265
#: ../../lib/entropy/const.py:637
msgid "All fine"
msgstr "Всё замечательно"

#: ../../lib/entropy/client/interfaces/methods.py:1226
msgid "Restoring backed up repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1244
msgid "Unable to unpack"
msgstr "Невозможно распаковать"

#: ../../lib/entropy/client/interfaces/methods.py:1254
msgid "Repository restored successfully"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1377
#: ../../lib/entropy/client/interfaces/package.py:4162
msgid "Removing"
msgstr "Удаление"

#: ../../lib/entropy/client/interfaces/methods.py:1820
msgid "Resources unlocked, let's go!"
msgstr "Ресурсы освобождены. Поехали!"

#: ../../lib/entropy/client/interfaces/methods.py:1828
msgid "Resources still locked, giving up!"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:1836
msgid "Resources locked, sleeping..."
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:2030
msgid "Checking speed of"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:2061
msgid "Mirror speed"
msgstr ""

#: ../../lib/entropy/client/interfaces/methods.py:2585
msgid "not a valid method"
msgstr "неправильный метод"

#: ../../lib/entropy/client/interfaces/package.py:372
msgid "Already prepared"
msgstr "Уже подготовлено"

#: ../../lib/entropy/client/interfaces/package.py:377
msgid "Not yet prepared"
msgstr "Ещё не подготовлено"

#: ../../lib/entropy/client/interfaces/package.py:609
#: ../../lib/entropy/client/interfaces/package.py:1280
msgid "Mirror"
msgstr "Зеркало"

#: ../../lib/entropy/client/interfaces/package.py:611
#: ../../lib/entropy/client/interfaces/package.py:1282
msgid "maximum failure threshold reached"
msgstr "Максимальный порог ошибок достигнут"

#: ../../lib/entropy/client/interfaces/package.py:656
msgid "success"
msgstr "успех"

#: ../../lib/entropy/client/interfaces/package.py:665
msgid "Aggregated transfer rate"
msgstr "Аггрегированная скорость передачи"

#: ../../lib/entropy/client/interfaces/package.py:668
#: ../../lib/entropy/client/interfaces/package.py:1338
#: ../../lib/entropy/client/interfaces/package.py:3457
msgid "second"
msgstr "сек."

#: ../../lib/entropy/client/interfaces/package.py:683
#: ../../lib/entropy/client/interfaces/package.py:1354
#: ../../lib/entropy/client/interfaces/package.py:3472
msgid "Error downloading from"
msgstr "Ошибка при скачивании с"

#: ../../lib/entropy/client/interfaces/package.py:687
msgid "data not available on this mirror"
msgstr "информация недоступна на этом зеркале"

#: ../../lib/entropy/client/interfaces/package.py:690
#: ../../lib/entropy/client/interfaces/package.py:1363
msgid "wrong checksum"
msgstr "неверная контрольная сумма"

#. timeout!
#: ../../lib/entropy/client/interfaces/package.py:694
msgid "timeout error"
msgstr "ошибка при превышении времени ожидания"

#: ../../lib/entropy/client/interfaces/package.py:696
#: ../../lib/entropy/client/interfaces/package.py:1384
#: ../../lib/entropy/client/interfaces/package.py:3483
msgid "discarded download"
msgstr "отмененная загрузка"

#: ../../lib/entropy/client/interfaces/package.py:699
#: ../../lib/entropy/client/interfaces/package.py:1387
#: ../../lib/entropy/client/interfaces/package.py:3485
msgid "unknown reason"
msgstr "неизвестная причина"

#: ../../lib/entropy/client/interfaces/package.py:1310
msgid "Downloading from"
msgstr "Скачивание с"

#: ../../lib/entropy/client/interfaces/package.py:1332
#: ../../lib/entropy/client/interfaces/package.py:3454
msgid "Successfully downloaded from"
msgstr "Успешно скачано с"

#: ../../lib/entropy/client/interfaces/package.py:1337
#: ../../lib/entropy/client/interfaces/package.py:3457
msgid "at"
msgstr "на"

#: ../../lib/entropy/client/interfaces/package.py:1360
#: ../../lib/entropy/client/interfaces/package.py:3478
msgid "file not available on this mirror"
msgstr "файл недоступен на этом зеркале"

#: ../../lib/entropy/client/interfaces/package.py:1368
msgid "Disabling resume"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1378
msgid "timeout, retrying on this mirror"
msgstr "превышено время ожидания, повтор попытки на данном зеркале"

#: ../../lib/entropy/client/interfaces/package.py:1381
msgid "timeout, giving up"
msgstr "превышено время ожидания, прекращаю"

#: ../../lib/entropy/client/interfaces/package.py:1486
msgid "Package signature verification error for"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1518
msgid "disabled"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1531
msgid "Checking package signature"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1542
msgid "Package signature verification"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1544
msgid "temporarily unavailable"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1554
msgid "Package signature"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1556
msgid "does not match the recorded one"
msgstr "не подсчитывает записанный"

#: ../../lib/entropy/client/interfaces/package.py:1566
msgid "matches"
msgstr "подсчёты"

#: ../../lib/entropy/client/interfaces/package.py:1581
msgid "Checking package checksum..."
msgstr "Подсчёт контрольной суммы пакета..."

#: ../../lib/entropy/client/interfaces/package.py:1594
msgid "Package checksum matches"
msgstr "Контрольная сумма пакета корректна"

#: ../../lib/entropy/client/interfaces/package.py:1614
#, python-format
msgid "Checksum does not match. Download attempt #%s"
msgstr "Контрольная сумма пакета некорректна. Попытка загрузки #%s"

#: ../../lib/entropy/client/interfaces/package.py:1636
msgid "Cannot properly fetch package! Quitting."
msgstr "В общем, невозможно загрузить пакет! Выход."

#: ../../lib/entropy/client/interfaces/package.py:1648
msgid "Cannot fetch package or checksum does not match"
msgstr "Невозможно загрузить пакет либо контрольная сумма некорректна."

#: ../../lib/entropy/client/interfaces/package.py:1649
msgid "Try to download latest repositories"
msgstr "Попробуйте загрузить последние репозитории"

#: ../../lib/entropy/client/interfaces/package.py:1678
#: ../../lib/entropy/client/interfaces/package.py:4134
msgid "Unpacking"
msgstr "Распаковка"

#: ../../lib/entropy/client/interfaces/package.py:1719
msgid "Unpack error"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1748
msgid "Unable to find Entropy metadata in package"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:1812
msgid "configuration phase"
msgstr "Фаза конфигурации"

#: ../../lib/entropy/client/interfaces/package.py:1865
msgid "Removing from Entropy"
msgstr "Removing from Entropy"

#: ../../lib/entropy/client/interfaces/package.py:1951
msgid "Removing config file, never modified"
msgstr "Удаление конфигурационного файла, никогда не модифицированного"

#: ../../lib/entropy/client/interfaces/package.py:1973
msgid "remove"
msgstr "удалить"

#: ../../lib/entropy/client/interfaces/package.py:1974
#: ../../lib/entropy/client/interfaces/package.py:3345
msgid "Protecting config file"
msgstr "Защита конфигурационного файла"

#: ../../lib/entropy/client/interfaces/package.py:1993
msgid "This package contains a badly encoded file !!!"
msgstr "Данный пакет содержит файл с испорченной кодировкой!!!"

#: ../../lib/entropy/client/interfaces/package.py:2110
msgid "Collision found during removal of"
msgstr "Произошла ошибка при удалении"

#: ../../lib/entropy/client/interfaces/package.py:2226
msgid "Updating database"
msgstr "Обновление базы данных"

#: ../../lib/entropy/client/interfaces/package.py:2720
#, python-format
msgid "%s is a file when should be a directory !! Removing in 20 seconds..."
msgstr "%s - это файл, который должен быть папкой !! Удаляется в течение 20 секунд..."

#: ../../lib/entropy/client/interfaces/package.py:2745
msgid "directory expected, symlink found"
msgstr "найден ярлык, хотя нужна была папка"

#: ../../lib/entropy/client/interfaces/package.py:2748
#: ../../lib/entropy/client/interfaces/package.py:3052
msgid "Removing in 20 seconds !!"
msgstr "Удаление через 20 секунд !!"

#: ../../lib/entropy/client/interfaces/package.py:2794
msgid "Cannot remove symlink"
msgstr ""

#. we can merge it, files, even if
#. contains changes have not been modified
#. by the user
#: ../../lib/entropy/client/interfaces/package.py:2958
msgid "Automerging config file, never modified"
msgstr "Автообъединение конфигурационного файла, никогда не модифицировавшегося"

#: ../../lib/entropy/client/interfaces/package.py:2987
#: ../../lib/entropy/client/interfaces/package.py:3009
msgid "Circular symlink issue"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3049
msgid "file expected, directory found"
msgstr "найдена папка, хотя нужен был файл"

#: ../../lib/entropy/client/interfaces/package.py:3102
msgid "File move error"
msgstr "Ошибка при перемещении файла"

#: ../../lib/entropy/client/interfaces/package.py:3105
msgid "please report"
msgstr "пожалуйста, отправьте отчет"

#: ../../lib/entropy/client/interfaces/package.py:3257
msgid "Cannot protect broken symlink"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3301
msgid "Skipping file installation/removal"
msgstr "Пропуск установки/удаления файла"

#: ../../lib/entropy/client/interfaces/package.py:3362
msgid "Collision found during install for"
msgstr "Обнаружено противоречие во время установки"

#: ../../lib/entropy/client/interfaces/package.py:3365
msgid "cannot overwrite"
msgstr "невозможно перезаписать"

#: ../../lib/entropy/client/interfaces/package.py:3438
#: ../../lib/entropy/client/interfaces/package.py:3501
#: ../../lib/entropy/client/interfaces/package.py:3548
#: ../../client/solo/utils.py:261 ../../client/solo/commands/command.py:316
msgid "Downloading"
msgstr "Скачивание"

#: ../../lib/entropy/client/interfaces/package.py:3465
msgid "Local path"
msgstr "Локальный путь"

#: ../../lib/entropy/client/interfaces/package.py:3531
msgid "Package cannot be fetched. Try to update repositories"
msgstr "Пакет не может быть загружен. Попробуйте обновить репозитории."

#: ../../lib/entropy/client/interfaces/package.py:3550
msgid "archive"
msgid_plural "archives"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/package.py:3567
msgid "Some packages cannot be fetched"
msgstr "Некоторые пакеты не могут быть загружены"

#: ../../lib/entropy/client/interfaces/package.py:3568
msgid "Try to update your repositories and retry"
msgstr "Попробуйте обновить репозитории и попробовать снова"

#: ../../lib/entropy/client/interfaces/package.py:3596
msgid "Package cannot be downloaded, unknown error."
msgstr "Пакет не может быть скачан, неизвестная ошибка."

#: ../../lib/entropy/client/interfaces/package.py:3605
msgid "Installed package in queue vanished, skipping."
msgstr "Установленные пакеты в очереди очищены, пропускаем."

#: ../../lib/entropy/client/interfaces/package.py:3641
msgid "Merging package"
msgstr "Соединение пакета"

#: ../../lib/entropy/client/interfaces/package.py:3694
msgid "You are running out of disk space"
msgstr "Осталось совсем мало места на диске"

#: ../../lib/entropy/client/interfaces/package.py:3695
msgid "I bet, you're probably Michele"
msgstr "Спорим, что ты Микеле"

#: ../../lib/entropy/client/interfaces/package.py:3699
msgid "An error occured while trying to unpack the package"
msgstr "Произошла ошибка при попытке распаковать пакет"

#: ../../lib/entropy/client/interfaces/package.py:3702
#: ../../lib/entropy/client/interfaces/package.py:3758
msgid "Check if your system is healthy"
msgstr "Проверьте, всё ли в порядке с системой"

#: ../../lib/entropy/client/interfaces/package.py:3721
msgid "Installing package"
msgstr "Установка пакета"

#: ../../lib/entropy/client/interfaces/package.py:3741
msgid "unsupported splitdebug usage (package files)"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3745
msgid "<3 debug files installation enabled <3"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3757
msgid "An error occured while trying to install the package"
msgstr "Произошла ошибка при попытке установить пакет"

#: ../../lib/entropy/client/interfaces/package.py:3772
msgid "Removing data"
msgstr "Удаление информации"

#: ../../lib/entropy/client/interfaces/package.py:3785
msgid "An error occured while trying to remove the package"
msgstr "Произошла ошибка при попытке удалить пакет"

#: ../../lib/entropy/client/interfaces/package.py:3786
msgid "Check if you have enough disk space on your hard disk"
msgstr "Проверьте, достаточно ли места на жёстком диске"

#: ../../lib/entropy/client/interfaces/package.py:3800
#: ../../lib/entropy/client/interfaces/package.py:4170
#: ../../client/solo/utils.py:64
msgid "Cleaning"
msgstr "Очистка"

#: ../../lib/entropy/client/interfaces/package.py:3873
msgid "Error during package files permissions setup"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:3909
msgid "Cleaning previously installed application data."
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:4020
msgid "Configuring package"
msgstr "Конфигурирование пакета"

#: ../../lib/entropy/client/interfaces/package.py:4032
#: ../../lib/entropy/client/interfaces/package.py:4052
msgid "An error occured while trying to configure the package"
msgstr "Случилась ошибка при попытке конфигурирования пакета"

#: ../../lib/entropy/client/interfaces/package.py:4034
msgid "Make sure that your system is healthy"
msgstr "Будьте уверены, что с Вашей системой всё в порядке"

#: ../../lib/entropy/client/interfaces/package.py:4054
msgid "It seems that Source Package Manager entry is missing"
msgstr "Это означает, что недостаёт записи Менеджера Пакетов Источника"

#: ../../lib/entropy/client/interfaces/package.py:4078
msgid "Installed package vanished"
msgstr "Установленный пакет исчез"

#: ../../lib/entropy/client/interfaces/package.py:4084
msgid "Fetch not available"
msgstr "Загрузка недоступна"

#: ../../lib/entropy/client/interfaces/package.py:4091
msgid "Fetching"
msgstr "Загрузка"

#: ../../lib/entropy/client/interfaces/package.py:4099
msgid "Multi Fetching"
msgstr "Мульти-загрузка"

#: ../../lib/entropy/client/interfaces/package.py:4100
#: ../../lib/entropy/client/interfaces/package.py:4120
#: ../../client/solo/commands/query.py:1272
#: ../../client/solo/commands/_manage.py:603
msgid "package"
msgid_plural "packages"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/package.py:4106
msgid "Fetching sources"
msgstr "Загрузка источников"

#: ../../lib/entropy/client/interfaces/package.py:4112
msgid "Verifying"
msgstr "Проверка"

#: ../../lib/entropy/client/interfaces/package.py:4119
msgid "Multi Verification"
msgstr "Мульти-проверка"

#: ../../lib/entropy/client/interfaces/package.py:4126
msgid "Merging"
msgstr "Объединение"

#: ../../lib/entropy/client/interfaces/package.py:4147
msgid "Installing"
msgstr "Установка"

#: ../../lib/entropy/client/interfaces/package.py:4178
msgid "Postinstall"
msgstr "Завершение установки"

#: ../../lib/entropy/client/interfaces/package.py:4186
msgid "Setup"
msgstr ""

#: ../../lib/entropy/client/interfaces/package.py:4194
msgid "Preinstall"
msgstr "Подготовка к установке"

#: ../../lib/entropy/client/interfaces/package.py:4202
msgid "Preremove"
msgstr "Подготовка к удалению"

#: ../../lib/entropy/client/interfaces/package.py:4210
msgid "Postremove"
msgstr "Завершение удаления"

#: ../../lib/entropy/client/interfaces/package.py:4224
msgid "Configuring"
msgstr "Конфигурирование"

#: ../../lib/entropy/client/interfaces/package.py:4277
msgid "An error occured. Action aborted."
msgstr "Произошла ошибка. Действие отменено."

#: ../../lib/entropy/client/interfaces/db.py:298
msgid "Injecting downloaded dump"
msgstr "Добавляется объём загруженного"

#: ../../lib/entropy/client/interfaces/db.py:397
msgid "Scanning URL"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:415
msgid "Selected URL"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:439
msgid "repository is not available at the database URLs"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:446
msgid "Looking for an alternate route using package mirrors"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:455
msgid "Checking repository URL"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:474
msgid "Found repository at URL"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:486
msgid "Unable to find alternate repository mirrors. Sorry."
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:508
#: ../../client/solo/commands/status.py:153
msgid "Repository URL"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:518
msgid "Repository local path"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:526
msgid "Repository API"
msgstr "API репозитория"

#: ../../lib/entropy/client/interfaces/db.py:576
msgid "Downloading repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:626
msgid "unable to download the repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:647
msgid "Downloading checksum"
msgstr "Загрузка контрольной суммы"

#: ../../lib/entropy/client/interfaces/db.py:674
msgid "Cannot fetch checksum"
msgstr "Невозможно загрузить контрольную сумму"

#: ../../lib/entropy/client/interfaces/db.py:675
#: ../../lib/entropy/client/interfaces/db.py:1247
msgid "Cannot verify repository integrity"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:770
msgid "Unpacking database to"
msgstr "Распаковка базы данных в"

#: ../../lib/entropy/client/interfaces/db.py:787
msgid "Cannot unpack compressed package"
msgstr "Невозможно распаковать сжатый пакет"

#: ../../lib/entropy/client/interfaces/db.py:788
msgid "Skipping repository"
msgstr "Пропуск репозитория"

#: ../../lib/entropy/client/interfaces/db.py:816
msgid "Repository is invalid"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:840
msgid "Indexing Repository metadata"
msgstr "Индексирование метаинформации репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1054
msgid "Downloading repository metafile"
msgstr "Скачивание метафайла репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1064
msgid "Downloading GPG signature of repository metafile"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1074
msgid "Downloading Notice Board"
msgstr "Загрузка доски уведомлений"

#: ../../lib/entropy/client/interfaces/db.py:1100
msgid "unpacked meta file"
msgstr "распакованный файл meta"

#: ../../lib/entropy/client/interfaces/db.py:1106
msgid "removed meta file"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1123
msgid "not available, it's ok"
msgstr "недоступно, но это нормально"

#: ../../lib/entropy/client/interfaces/db.py:1127
msgid "not available, not very ok!"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1140
msgid "available, w00t!"
msgstr "ура, доступно!"

#: ../../lib/entropy/client/interfaces/db.py:1158
msgid "cannot be unpacked, not very ok!"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1209
msgid "Repository revision"
msgstr "Версия репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1232
msgid "Checking downloaded repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1246
msgid "Cannot open digest"
msgstr "Невозможно открыть обзор"

#: ../../lib/entropy/client/interfaces/db.py:1257
#: ../../lib/entropy/client/interfaces/db.py:1268
msgid "Downloaded repository status"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1258
msgid "OK"
msgstr "OK"

#: ../../lib/entropy/client/interfaces/db.py:1269
msgid "ERROR"
msgstr "ОШИБКА"

#: ../../lib/entropy/client/interfaces/db.py:1278
msgid "An error occured while checking repository integrity"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1279
msgid "Giving up"
msgstr "Прекращаю"

#: ../../lib/entropy/client/interfaces/db.py:1301
#: ../../lib/entropy/security.py:283
msgid ""
"Make sure to verify the imported key and set an appropriate trust level"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1321
msgid "This repository suports GPG-signed packages"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1328
msgid "you may want to install GnuPG to take advantage of this feature"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1363
#: ../../lib/entropy/security.py:322
msgid "GPG key changed for"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1383
#: ../../lib/entropy/security.py:341
msgid "GPG key already installed for"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1396
msgid "GPG key EXPIRED for repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1408
msgid "Installing GPG key for repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1426
#: ../../lib/entropy/client/interfaces/db.py:1459
#: ../../lib/entropy/security.py:379
msgid "Error during GPG key installation"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1436
msgid "GPG key seems already installed but not properly recorded, resetting"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1471
msgid "Successfully installed GPG key for repository"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1480
#: ../../lib/entropy/security.py:399
msgid "Fingerprint"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1520
msgid "Verifying GPG signature of"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1534
msgid "Verified GPG signature of"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1544
#: ../../lib/entropy/security.py:424
msgid "Error during GPG verification of"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1553
#: ../../lib/entropy/security.py:433
msgid "It could mean a potential security risk"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1611
msgid "Web Service"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1612
msgid "skipping differential sync"
msgstr "пропуск различной синхронизации"

#: ../../lib/entropy/client/interfaces/db.py:1613
msgid "threshold"
msgstr "порог"

#: ../../lib/entropy/client/interfaces/db.py:1647
msgid "Web Service communication error"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1667
msgid "Web Service data error"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1683
msgid "Local status"
msgstr "Локальный статус"

#: ../../lib/entropy/client/interfaces/db.py:1715
msgid "Fetching segments"
msgstr "Загрузка сегментов"

#: ../../lib/entropy/client/interfaces/db.py:1751
#: ../../lib/entropy/client/interfaces/db.py:1770
#: ../../lib/entropy/client/interfaces/db.py:1787
msgid "Web Service status"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1752
msgid "cannot fetch repository metadata"
msgstr "невозможно загрузить метаинформацию репозитория"

#: ../../lib/entropy/client/interfaces/db.py:1771
msgid "cannot update treeupdates data"
msgstr "невозможно обновить treeupdates-информацию"

#: ../../lib/entropy/client/interfaces/db.py:1788
msgid "cannot update package sets data"
msgstr "невозможно обновить информацию по наборам пакетов"

#: ../../lib/entropy/client/interfaces/db.py:1804
msgid "Fetch error on segment while adding"
msgstr "Ошибка загрузки сегмента при добавлении"

#: ../../lib/entropy/client/interfaces/db.py:1830
msgid "repository error while adding packages"
msgstr "Ошибка репозитория при добавлении пакетов"

#: ../../lib/entropy/client/interfaces/db.py:1853
msgid "repository error while removing packages"
msgstr "ошибка репозитория при удалении пакетов"

#: ../../lib/entropy/client/interfaces/db.py:1869
msgid "Repository checksum doesn't match remote."
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:1872
msgid "local"
msgstr "локально"

#: ../../lib/entropy/client/interfaces/db.py:1877
msgid "remote"
msgstr "отдаленное"

#: ../../lib/entropy/client/interfaces/db.py:1987
msgid "repository is already up to date"
msgstr ""

#: ../../lib/entropy/client/interfaces/db.py:2000
msgid "Repository is being updated"
msgstr "Репозиторий обновляется"

#: ../../lib/entropy/client/interfaces/db.py:2001
msgid "Try again in a few minutes"
msgstr "Попробуйте снова через несколько минут"

#: ../../lib/entropy/client/interfaces/db.py:2194
msgid "Configuration files update error, not critical, continuing"
msgstr "Ошибка при обновлении файлов конфигурации, некритично, продолжаем"

#: ../../lib/entropy/client/interfaces/repository.py:173
msgid "Something bad happened. Please have a look."
msgstr "Случилось что-то нехорошее. Пожалуйста, взгляните."

#: ../../lib/entropy/client/interfaces/repository.py:195
msgid "there is"
msgid_plural "there are"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/repository.py:197
msgid "package file that could be removed"
msgid_plural "package files that could be removed"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/client/interfaces/repository.py:210
msgid "Packages are stored in"
msgstr ""

#: ../../lib/entropy/client/interfaces/repository.py:240
msgid "a new release is available"
msgstr "доступна новая версия"

#: ../../lib/entropy/client/interfaces/repository.py:241
msgid "Mind to install it before any other package"
msgstr "Не забудьте установить это перед любыми другими пакетами"

#: ../../lib/entropy/client/interfaces/repository.py:306
msgid "Advisories fetch error"
msgstr "Ошибка при загрузке уведомлений"

#. let's dance!
#: ../../lib/entropy/client/interfaces/repository.py:338
msgid "Repositories synchronization"
msgstr "Синхронизация репозитория"

#: ../../lib/entropy/client/interfaces/sets.py:238
msgid "Name already taken"
msgstr "Имя уже установлено"

#: ../../lib/entropy/client/interfaces/sets.py:247
msgid "Cannot remove the old element"
msgstr "Невозможно удалить старый элемент"

#: ../../lib/entropy/client/interfaces/sets.py:249
msgid "Cannot create the element"
msgstr "Невозможно создать элемент"

#: ../../lib/entropy/client/interfaces/sets.py:276
msgid "cannot start with"
msgstr "невозможно стартовать с"

#: ../../lib/entropy/client/interfaces/sets.py:281
msgid "Already removed"
msgstr "Уже удалено"

#: ../../lib/entropy/client/interfaces/sets.py:285
msgid "Not defined by user"
msgstr "Не указано пользователем"

#: ../../lib/entropy/client/interfaces/sets.py:293
msgid "Set not found or unable to remove"
msgstr "Набор не найден или невозможно удалить"

#: ../../lib/entropy/client/interfaces/dep.py:53
#, python-format
msgid "Checking %s"
msgstr "Проверка %s"

#: ../../lib/entropy/client/interfaces/dep.py:2186
msgid "Sorting dependencies"
msgstr "Сортировка зависимостей"

#: ../../lib/entropy/client/interfaces/dep.py:2391
msgid "Calculating inverse dependencies for"
msgstr "Подсчет противоположных зависимостей для"

#: ../../lib/entropy/client/interfaces/dep.py:3041
msgid "Calculating updates"
msgstr ""

#: ../../lib/entropy/fetchers.py:807
msgid "infinite"
msgstr "бесконечно"

#: ../../lib/entropy/fetchers.py:879
msgid "[F]"
msgstr "[F]"

#: ../../lib/entropy/fetchers.py:880 ../../lib/entropy/fetchers.py:1250
msgid "ETA"
msgstr "ЕТА"

#: ../../lib/entropy/fetchers.py:881 ../../lib/entropy/fetchers.py:1251
msgid "sec"
msgstr "сек"

#: ../../lib/entropy/fetchers.py:1132
msgid "Aggregated download"
msgstr "Аггрегированная загрузка"

#: ../../lib/entropy/fetchers.py:1134
msgid "item"
msgid_plural "items"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../lib/entropy/security.py:354
msgid "GPG key EXPIRED for URL"
msgstr ""

#: ../../lib/entropy/security.py:365
msgid "Installing GPG key for URL"
msgstr ""

#: ../../lib/entropy/security.py:390
msgid "Successfully installed GPG key for URL"
msgstr ""

#: ../../lib/entropy/security.py:444 ../../lib/entropy/security.py:1019
#: ../../lib/entropy/security.py:1034 ../../lib/entropy/security.py:1041
#: ../../lib/entropy/security.py:1083 ../../lib/entropy/security.py:1106
#: ../../lib/entropy/security.py:1118 ../../lib/entropy/security.py:1135
#: ../../lib/entropy/security.py:1147 ../../lib/entropy/security.py:1159
#: ../../lib/entropy/security.py:1171 ../../lib/entropy/security.py:1192
#: ../../lib/entropy/security.py:1220 ../../lib/entropy/security.py:1232
msgid "Security Advisories"
msgstr "Предостережения по безопасности"

#: ../../lib/entropy/security.py:445
msgid "GPG key verification successful"
msgstr ""

#: ../../lib/entropy/security.py:661 ../../client/solo/commands/pkg.py:197
#: ../../client/solo/commands/_manage.py:456
#: ../../client/solo/commands/_manage.py:478
#: ../../client/solo/commands/mask.py:99
#: ../../client/solo/commands/mask.py:172
msgid "Warning"
msgstr "Осторожно"

#: ../../lib/entropy/security.py:663
msgid "advisory broken"
msgstr "уведомление испорчено"

#: ../../lib/entropy/security.py:1020
msgid "getting latest advisories"
msgstr ""

#: ../../lib/entropy/security.py:1035
msgid "updated successfully"
msgstr "успешно обновлено"

#: ../../lib/entropy/security.py:1042
msgid "already up to date"
msgstr "уже обновлено до последней версии"

#: ../../lib/entropy/security.py:1084
msgid "cannot download checksum, sorry"
msgstr ""

#: ../../lib/entropy/security.py:1107
msgid "unable to download advisories, sorry"
msgstr ""

#: ../../lib/entropy/security.py:1119
msgid "Verifying checksum"
msgstr "Проверка контрольной суммы"

#: ../../lib/entropy/security.py:1136
msgid "cannot open packages, sorry"
msgstr "увы, невозможно открыть пакеты"

#: ../../lib/entropy/security.py:1148
msgid "cannot read the checksum, sorry"
msgstr "жаль, но невозможно прочитать контрольную сумму"

#: ../../lib/entropy/security.py:1160
msgid "digest verification failed, sorry"
msgstr "Не удалось проверить списки, увы"

#: ../../lib/entropy/security.py:1172
msgid "verification successful"
msgstr ""

#: ../../lib/entropy/security.py:1193
msgid "GPG service not available"
msgstr ""

#: ../../lib/entropy/security.py:1221
msgid "digest verification failed, try again later"
msgstr "проверка списков не удалась, попробуйте позже"

#: ../../lib/entropy/security.py:1233
msgid "installing"
msgstr "установка"

#: ../../lib/entropy/const.py:615
msgid "Runtime dependency"
msgstr ""

#: ../../lib/entropy/const.py:616
msgid "Post dependency"
msgstr ""

#: ../../lib/entropy/const.py:617
msgid "Manually added (by staff) dependency"
msgstr ""

#: ../../lib/entropy/const.py:618
msgid "Build dependency"
msgstr ""

#: ../../lib/entropy/const.py:638
msgid "Corrupted Client Repository. Please restore a backup."
msgstr "Испорчен клиент репозитория. Пожалуйста, восстановите из резервной копии."

#: ../../lib/entropy/const.py:670
msgid "reason not available"
msgstr "причина недоступна"

#: ../../lib/entropy/const.py:671
msgid "user package.mask"
msgstr "package.mask пользователя"

#: ../../lib/entropy/const.py:672
msgid "system keywords"
msgstr "системные ключевые слова"

#: ../../lib/entropy/const.py:673
msgid "user package.unmask"
msgstr "package.unmask пользователя"

#: ../../lib/entropy/const.py:674
msgid "user repo package.keywords (all packages)"
msgstr "user repo package.keywords (все пакеты)"

#: ../../lib/entropy/const.py:675
msgid "user repo package.keywords"
msgstr "user repo package.keywords"

#: ../../lib/entropy/const.py:676
msgid "user package.keywords"
msgstr "package.keywords пользователя"

#: ../../lib/entropy/const.py:677
msgid "completely masked (by keyword?)"
msgstr "полностью замаскировано (по ключевому слову?)"

#: ../../lib/entropy/const.py:678
msgid "repository general packages.db.mask"
msgstr "repository general packages.db.mask"

#: ../../lib/entropy/const.py:679
msgid "repository general packages.db.keywords"
msgstr "repository in branch packages.db.mask"

#: ../../lib/entropy/const.py:680
msgid "user license.mask"
msgstr "user license.mask"

#: ../../lib/entropy/const.py:681
msgid "user live unmask"
msgstr "user live unmask"

#: ../../lib/entropy/const.py:682
msgid "user live mask"
msgstr "user live mask"

#: ../../lib/entropy/output.py:628
msgid "Entropy needs your attention"
msgstr "Entropy требует Вашего внимания"

#: ../../lib/entropy/output.py:766 ../../lib/entropy/output.py:1011
#: ../../client/solo/utils.py:432 ../../client/solo/commands/ugc.py:652
#: ../../client/solo/commands/ugc.py:762 ../../client/solo/commands/ugc.py:785
#: ../../client/solo/commands/ugc.py:966
#: ../../client/solo/commands/upgrade.py:292
#: ../../client/solo/commands/upgrade.py:320
#: ../../client/solo/commands/rescue.py:666
#: ../../client/solo/commands/rescue.py:671
#: ../../client/solo/commands/rescue.py:711
msgid "Yes"
msgstr "Да"

#: ../../lib/entropy/output.py:766 ../../client/solo/utils.py:434
#: ../../client/solo/main.py:168 ../../client/solo/commands/pkg.py:255
#: ../../client/solo/commands/config.py:149
#: ../../client/solo/commands/remove.py:398
#: ../../client/solo/commands/remove.py:405
#: ../../client/solo/commands/remove.py:449
#: ../../client/solo/commands/deptest.py:182
#: ../../client/solo/commands/libtest.py:223
#: ../../client/solo/commands/_manage.py:241
#: ../../client/solo/commands/conf.py:391
#: ../../client/solo/commands/conf.py:446
#: ../../client/solo/commands/install.py:554
#: ../../client/solo/commands/upgrade.py:292
#: ../../client/solo/commands/rescue.py:328
#: ../../client/solo/commands/rescue.py:332
#: ../../client/solo/commands/rescue.py:339
#: ../../client/solo/commands/mask.py:161
msgid "No"
msgstr "Нет"

#: ../../lib/entropy/output.py:781
msgid "Entropy got a question for you"
msgstr "у Entropy есть к Вам вопросы"

#: ../../lib/entropy/output.py:795
msgid "Interrupted"
msgstr "Отменено"

#: ../../lib/entropy/output.py:857 ../../lib/entropy/output.py:861
msgid "Selected number"
msgstr "Выбранный номер"

#: ../../lib/entropy/output.py:878 ../../client/solo/commands/_manage.py:715
msgid "Please select an option"
msgstr "Пожалуйста, выберите опцию"

#: ../../lib/entropy/output.py:880
msgid "Discard all"
msgstr "Отменить всё"

#: ../../lib/entropy/output.py:881
msgid "Confirm"
msgstr "Подтвердить"

#: ../../lib/entropy/output.py:882
msgid "Add item"
msgstr "Добавить предмет"

#: ../../lib/entropy/output.py:883
msgid "Edit item"
msgstr ""

#: ../../lib/entropy/output.py:884
msgid "Remove item"
msgstr "Удалить предмет"

#: ../../lib/entropy/output.py:885
msgid "Show current list"
msgstr "Показать текущий список"

#: ../../lib/entropy/output.py:889 ../../client/solo/commands/conf.py:247
#: ../../client/solo/commands/conf.py:290
msgid "Your choice (type a number and press enter):"
msgstr "Ваш выбор (введите номер и нажмите Enter)"

#: ../../lib/entropy/output.py:921
msgid "You don't have typed a number."
msgstr "Вы не ввели номер."

#: ../../lib/entropy/output.py:924
msgid "Invalid action."
msgstr "Неправильное действие."

#: ../../lib/entropy/output.py:934
msgid "String to add (-1 to go back):"
msgstr ""

#: ../../lib/entropy/output.py:944
msgid "Invalid string."
msgstr "Неправильная строка."

#: ../../lib/entropy/output.py:952
msgid "Element number to edit (-1 to go back):"
msgstr ""

#: ../../lib/entropy/output.py:963
msgid "old"
msgstr ""

#: ../../lib/entropy/output.py:963
msgid "new value:"
msgstr ""

#: ../../lib/entropy/output.py:971 ../../lib/entropy/output.py:989
msgid "Invalid element."
msgstr "Неправильный элемент."

#: ../../lib/entropy/output.py:980
msgid "Element number to remove (-1 to go back):"
msgstr ""

#: ../../lib/entropy/qa.py:161
msgid "Searching for broken reverse dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:178
msgid "scanning for broken reverse dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:219
msgid "broken libraries detected"
msgstr "испорченные библиотеки найдены"

#: ../../lib/entropy/qa.py:230
msgid "needs"
msgstr "требует"

#: ../../lib/entropy/qa.py:269
msgid "scanning"
msgstr ""

#: ../../lib/entropy/qa.py:311
msgid "blacklisted dependencies !!!"
msgstr ""

#: ../../lib/entropy/qa.py:336
msgid "requires libraries"
msgstr ""

#: ../../lib/entropy/qa.py:354
msgid "library provided by"
msgstr ""

#: ../../lib/entropy/qa.py:400 ../../lib/entropy/qa.py:627
msgid "no packages"
msgstr ""

#: ../../lib/entropy/qa.py:429
msgid "is potentially missing these dependencies"
msgstr ""

#: ../../lib/entropy/qa.py:566
msgid "determining missing libraries"
msgstr ""

#: ../../lib/entropy/qa.py:598
msgid "requires"
msgstr ""

#: ../../lib/entropy/qa.py:615 ../../lib/entropy/qa.py:626
msgid "provided by"
msgstr ""

#: ../../lib/entropy/qa.py:639
msgid "no missing runtime libraries found"
msgstr ""

#: ../../lib/entropy/qa.py:675
msgid "Libraries test"
msgstr "Тест библиотек"

#: ../../lib/entropy/qa.py:690
msgid "Broken symbols packages list"
msgstr "Список испорченных пакетов символов"

#: ../../lib/entropy/qa.py:691
msgid "Broken executables list"
msgstr "Список испорченных исполняемых файлов"

#: ../../lib/entropy/qa.py:693
msgid "Dumping results into these files"
msgstr "Запись результатов в эти файлы"

#: ../../lib/entropy/qa.py:773
msgid "discarding directory"
msgstr "папка забракованного"

#: ../../lib/entropy/qa.py:775
msgid "because it's symlinked on"
msgstr "поскольку оно ссылается на"

#: ../../lib/entropy/qa.py:837
msgid "Collecting broken executables"
msgstr "Ищутся испорченные исполняемые файлы"

#: ../../lib/entropy/qa.py:843
msgid "don't worry about libraries that are shown here but not later."
msgstr "не беспокойтесь о библиотеках, которые показаны здесь, но не позже"

#: ../../lib/entropy/qa.py:864
msgid "Scanning libraries"
msgstr "Сканирование библиотек"

#: ../../lib/entropy/qa.py:972
msgid "various broken symbols"
msgstr "Различные испорченные символы"

#: ../../lib/entropy/qa.py:1013
msgid "Matching broken libraries/executables"
msgstr "Подсчет испорченных библиотек/исполняемых приложений"

#: ../../lib/entropy/qa.py:1738
msgid "Not prepared yet"
msgstr "Ещё не подготовлено."

#: ../../client/solo/utils.py:65
msgid "directory"
msgstr ""

#: ../../client/solo/utils.py:73
msgid "Cleaned"
msgstr ""

#: ../../client/solo/utils.py:75
msgid "files and directories"
msgstr ""

#: ../../client/solo/utils.py:184 ../../client/solo/utils.py:327
#: ../../client/solo/utils.py:328 ../../client/solo/utils.py:424
#: ../../client/solo/utils.py:426 ../../client/solo/commands/repo.py:501
#: ../../client/solo/commands/repo.py:556
msgid "N/A"
msgstr "N/A"

#: ../../client/solo/utils.py:263 ../../client/solo/commands/command.py:318
msgid "Uploading"
msgstr ""

#: ../../client/solo/utils.py:285 ../../client/solo/commands/rescue.py:235
msgid "corrupted"
msgstr ""

#: ../../client/solo/utils.py:326 ../../client/solo/commands/_manage.py:149
#: ../../client/solo/commands/_manage.py:193
msgid "Not installed"
msgstr "Не установлено"

#: ../../client/solo/utils.py:352 ../../client/solo/commands/query.py:460
#: ../../client/solo/commands/query.py:703
#: ../../client/solo/commands/query.py:1028
msgid "Package"
msgstr "Пакет"

#: ../../client/solo/utils.py:354 ../../client/solo/commands/hop.py:57
msgid "branch"
msgstr "ветка"

#: ../../client/solo/utils.py:361
msgid "Category"
msgstr "Категория"

#: ../../client/solo/utils.py:363
msgid "Name"
msgstr "Имя"

#: ../../client/solo/utils.py:379 ../../client/solo/commands/query.py:568
msgid "Masked"
msgstr "Замаскировано"

#: ../../client/solo/utils.py:382
msgid "Available"
msgstr "Доступно"

#: ../../client/solo/utils.py:384 ../../client/solo/utils.py:392
msgid "Installed"
msgstr "Установлено"

#: ../../client/solo/utils.py:387 ../../client/solo/utils.py:393
msgid "version"
msgstr "версия"

#: ../../client/solo/utils.py:388 ../../client/solo/utils.py:395
msgid "revision"
msgstr "версия"

#: ../../client/solo/utils.py:398
msgid "Slot"
msgstr "Слот"

#: ../../client/solo/utils.py:415
msgid "Size"
msgstr "Размер"

#: ../../client/solo/utils.py:417 ../../client/solo/commands/ugc.py:378
msgid "Download"
msgstr "Загрузка"

#: ../../client/solo/utils.py:419
msgid "Checksum"
msgstr "Контрольная сумма"

#: ../../client/solo/utils.py:427
msgid "SHA1"
msgstr ""

#: ../../client/solo/utils.py:429
msgid "SHA256"
msgstr ""

#: ../../client/solo/utils.py:435 ../../client/solo/commands/status.py:133
msgid "GPG"
msgstr ""

#: ../../client/solo/utils.py:440
msgid "Dependencies"
msgstr "Зависимости"

#: ../../client/solo/utils.py:449 ../../client/solo/utils.py:632
msgid "Legend"
msgstr ""

#: ../../client/solo/utils.py:457
msgid "Conflicts"
msgstr "Конфликты"

#: ../../client/solo/utils.py:462
msgid "Homepage"
msgstr "Домашняя страница"

#: ../../client/solo/utils.py:472 ../../client/solo/commands/security.py:224
#: ../../client/solo/commands/ugc.py:623
msgid "Description"
msgstr "Описание"

#: ../../client/solo/utils.py:481
msgid "USE flags"
msgstr "USE-флаги"

#: ../../client/solo/utils.py:501
msgid "CHOST"
msgstr "CHOST"

#: ../../client/solo/utils.py:503
msgid "CFLAGS"
msgstr "CFLAGS"

#: ../../client/solo/utils.py:505
msgid "CXXFLAGS"
msgstr "CXXFLAGS"

#: ../../client/solo/utils.py:509
msgid "Sources"
msgstr "Источники"

#: ../../client/solo/utils.py:514
msgid "Entry API"
msgstr "Запись API"

#: ../../client/solo/utils.py:516
msgid "Compiled with"
msgstr "Компилировано с"

#: ../../client/solo/utils.py:521 ../../client/solo/commands/search.py:178
#: ../../client/solo/commands/ugc.py:371 ../../client/solo/commands/ugc.py:630
#: ../../client/solo/commands/match.py:219
msgid "Keywords"
msgstr "Ключевые слова"

#: ../../client/solo/utils.py:536
msgid "Created"
msgstr "Создано"

#: ../../client/solo/utils.py:542 ../../client/solo/commands/_manage.py:774
msgid "License"
msgstr "Лицензия"

#: ../../client/solo/utils.py:564
msgid "Reverse graphing installed package"
msgstr ""

#: ../../client/solo/utils.py:576 ../../client/solo/utils.py:735
#: ../../client/solo/commands/config.py:116
#: ../../client/solo/commands/remove.py:379
#: ../../client/solo/commands/source.py:161
#: ../../client/solo/commands/install.py:524
#: ../../client/solo/commands/download.py:161
#: ../../client/solo/commands/mask.py:118
msgid "No packages found"
msgstr "Пакетов не найдено"

#: ../../client/solo/utils.py:635
msgid "packages passed as arguments"
msgstr ""

#: ../../client/solo/utils.py:638
msgid "packages with no further dependencies"
msgstr ""

#: ../../client/solo/utils.py:641
msgid "packages with further dependencies (node)"
msgstr ""

#: ../../client/solo/utils.py:644
msgid "packages already pulled in as dependency in upper levels (circularity)"
msgstr ""

#: ../../client/solo/utils.py:723
msgid "Graphing"
msgstr ""

#: ../../client/solo/main.py:40
msgid "Installed packages repository corrupted. Please re-generate it"
msgstr ""

#: ../../client/solo/main.py:50
msgid "Cannot continue"
msgstr "Невозможно продолжить"

#: ../../client/solo/main.py:79
msgid "Your hard drive is full! Your fault!"
msgstr ""

#: ../../client/solo/main.py:88
msgid "No more memory dude! Your fault!"
msgstr ""

#: ../../client/solo/main.py:94
msgid ""
"Hi. My name is Bug Reporter. I am sorry to inform you that the program "
"crashed. Well, you know, shit happens."
msgstr ""

#: ../../client/solo/main.py:100
msgid ""
"But there's something you could do to help me to be a better application."
msgstr ""

#: ../../client/solo/main.py:106
msgid "-- BUT, DO NOT SUBMIT THE SAME REPORT MORE THAN ONCE --"
msgstr ""

#: ../../client/solo/main.py:111
msgid "Now I am showing you what happened. Don't panic, I'm here to help you."
msgstr "Теперь я покажу тебе, что случилось. Не паникуй, я здесь, чтобы помочь тебе. "

#: ../../client/solo/main.py:148
msgid ""
"Oh well, I cannot even write to TMPDIR. So, please copy the error and mail "
"lxnay@sabayon.org."
msgstr ""

#: ../../client/solo/main.py:163
msgid ""
"Erm... Can I send the error, along with some other information\n"
"about your hardware to my creators so they can fix me? (Your IP will be logged)"
msgstr ""

#: ../../client/solo/main.py:170
msgid "Ok, ok ok ok... Sorry!"
msgstr "Ну ладно, ладно, ладно, хорошо, ХОРОШО, ИЗВИНИ!"

#: ../../client/solo/main.py:176
msgid ""
"If you want to be contacted back (and actively supported), also answer the "
"questions below:"
msgstr "Когда будешь связываться (и активно общаться с техподдержкой), ответь заодно на нижеуказанные вопросы:"

#: ../../client/solo/main.py:183
msgid "Your Full name:"
msgstr "Ваше полное имя:"

#: ../../client/solo/main.py:184
msgid "Your E-Mail address:"
msgstr "Ваш адрес E-Mail:"

#: ../../client/solo/main.py:185
msgid "What you were doing:"
msgstr "Чего ж ты такого наделал:"

#: ../../client/solo/main.py:207
msgid ""
"Thank you very much. The error has been reported and hopefully, the problem "
"will be solved as soon as possible."
msgstr "Спасибо тебе огромное. Данные об ошибке отправлены и, если всё будет нормально, проблема будет решена как можно раньше."

#: ../../client/solo/main.py:213
msgid ""
"Ugh. Cannot send the report. Please mail the file below to "
"lxnay@sabayon.org."
msgstr ""

#: ../../client/solo/main.py:233
msgid "Entropy/Equo version mismatch"
msgstr ""

#: ../../client/solo/main.py:234
msgid "it could make your system explode!"
msgstr ""

#: ../../client/solo/main.py:243
msgid "Entropy is running off a Live System"
msgstr ""

#: ../../client/solo/main.py:245
msgid "Performance and stability could get severely compromised"
msgstr ""

#: ../../client/solo/main.py:347 ../kernel-switcher:114
msgid "superuser access required"
msgstr ""

#: ../../client/solo/commands/pkg.py:71 ../../client/solo/commands/pkg.py:484
#: ../../client/solo/commands/rescue.py:71
msgid "execute advanced tasks on packages"
msgstr ""

#: ../../client/solo/commands/pkg.py:72 ../../client/solo/commands/query.py:73
#: ../../client/solo/commands/query.py:154
#: ../../client/solo/commands/security.py:69
#: ../../client/solo/commands/cache.py:61 ../../client/solo/commands/ugc.py:72
#: ../../client/solo/commands/ugc.py:105 ../../client/solo/commands/ugc.py:146
#: ../../client/solo/commands/repo.py:68 ../../client/solo/commands/conf.py:76
#: ../../client/solo/commands/rescue.py:72 ../kernel-switcher:48
msgid "available commands"
msgstr ""

#: ../../client/solo/commands/pkg.py:78
#: ../../client/solo/commands/config.py:63
#: ../../client/solo/commands/security.py:90
#: ../../client/solo/commands/security.py:162
#: ../../client/solo/commands/remove.py:67
#: ../../client/solo/commands/deptest.py:61
#: ../../client/solo/commands/libtest.py:61
#: ../../client/solo/commands/source.py:62
#: ../../client/solo/commands/install.py:70
#: ../../client/solo/commands/upgrade.py:65
#: ../../client/solo/commands/rescue.py:78
#: ../../client/solo/commands/download.py:62
#: ../../client/solo/commands/mask.py:49
msgid "ask before making any changes"
msgstr "спрашивать перед любыми изменениями"

#: ../../client/solo/commands/pkg.py:85
#: ../../client/solo/commands/command.py:111
msgid "not a valid Entropy package file"
msgstr ""

#: ../../client/solo/commands/pkg.py:92
msgid "generate packages from the installed system"
msgstr ""

#: ../../client/solo/commands/pkg.py:96
msgid "installed package name"
msgstr ""

#: ../../client/solo/commands/pkg.py:101 ../../client/solo/commands/pkg.py:118
#: ../../client/solo/commands/pkg.py:135 ../../client/solo/commands/pkg.py:152
msgid "destination directory where to save generated packages"
msgstr ""

#: ../../client/solo/commands/pkg.py:108
msgid "transform SPM package files into Entropy ones"
msgstr ""

#: ../../client/solo/commands/pkg.py:113
msgid "SPM package file path"
msgstr ""

#: ../../client/solo/commands/pkg.py:125
msgid "transform Entropy package files into SPM ones"
msgstr ""

#: ../../client/solo/commands/pkg.py:130 ../../client/solo/commands/pkg.py:147
msgid "Entropy package file path"
msgstr ""

#: ../../client/solo/commands/pkg.py:142
msgid "extract Entropy metadata from Entropy packages"
msgstr ""

#: ../../client/solo/commands/pkg.py:225 ../../client/solo/commands/pkg.py:303
#: ../../client/solo/commands/pkg.py:399 ../../client/solo/commands/pkg.py:446
msgid "broken directory path"
msgstr ""

#: ../../client/solo/commands/pkg.py:238
msgid "This is the list of packages that would be considered"
msgstr ""

#: ../../client/solo/commands/pkg.py:254
#: ../../client/solo/commands/config.py:148
#: ../../client/solo/commands/_manage.py:240
#: ../../client/solo/commands/install.py:553
msgid "Would you like to continue ?"
msgstr "Желаете ли продолжить?"

#: ../../client/solo/commands/pkg.py:265
msgid "generating package"
msgstr ""

#: ../../client/solo/commands/pkg.py:275
msgid "package file creation error"
msgstr ""

#: ../../client/solo/commands/pkg.py:283
msgid "package generated"
msgstr ""

#: ../../client/solo/commands/pkg.py:312 ../../client/solo/commands/pkg.py:406
#: ../../client/solo/commands/pkg.py:453
msgid "working on package file"
msgstr ""

#: ../../client/solo/commands/pkg.py:325
msgid "package file extraction complete"
msgstr ""

#: ../../client/solo/commands/pkg.py:368
msgid "package metadata generation complete"
msgstr ""

#: ../../client/solo/commands/pkg.py:379
msgid "package file generated at"
msgstr ""

#: ../../client/solo/commands/pkg.py:418 ../../client/solo/commands/pkg.py:466
msgid "error during metadata extraction"
msgstr ""

#: ../../client/solo/commands/pkg.py:426
msgid "package file generated"
msgstr ""

#: ../../client/solo/commands/pkg.py:474
msgid "metadata file generated"
msgstr ""

#: ../../client/solo/commands/query.py:72
#: ../../client/solo/commands/query.py:1585
msgid "repository query tools"
msgstr ""

#: ../../client/solo/commands/query.py:77
msgid "resolve what package a file belongs to"
msgstr ""

#: ../../client/solo/commands/query.py:80
#: ../../client/solo/commands/query.py:200
msgid "file path"
msgstr ""

#: ../../client/solo/commands/query.py:87
msgid "show package changelog"
msgstr ""

#: ../../client/solo/commands/query.py:90
#: ../../client/solo/commands/query.py:100
#: ../../client/solo/commands/query.py:120
#: ../../client/solo/commands/query.py:130
#: ../../client/solo/commands/query.py:215
#: ../../client/solo/commands/query.py:282
#: ../../client/solo/commands/query.py:299
#: ../../client/solo/commands/config.py:57
#: ../../client/solo/commands/remove.py:61
#: ../../client/solo/commands/source.py:56
#: ../../client/solo/commands/install.py:64
#: ../../client/solo/commands/download.py:56
#: ../../client/solo/commands/mask.py:44
msgid "package name"
msgstr ""

#: ../../client/solo/commands/query.py:97
msgid "show reverse dependencies of package"
msgstr ""

#: ../../client/solo/commands/query.py:107
msgid "search package by description"
msgstr ""

#: ../../client/solo/commands/query.py:110
msgid "description keyword"
msgstr ""

#: ../../client/solo/commands/query.py:117
msgid "show files owned by package"
msgstr ""

#: ../../client/solo/commands/query.py:127
msgid "search installed packages"
msgstr ""

#: ../../client/solo/commands/query.py:137
msgid "show packages using the given license"
msgstr ""

#: ../../client/solo/commands/query.py:140
msgid "license name"
msgstr ""

#: ../../client/solo/commands/query.py:146
#: ../../client/solo/commands/query.py:153
msgid "list packages"
msgstr ""

#: ../../client/solo/commands/query.py:157
msgid "list installed packages"
msgstr "показать установленные пакеты"

#: ../../client/solo/commands/query.py:160
msgid "only list packages installed by user"
msgstr ""

#: ../../client/solo/commands/query.py:163
msgid "only list packages installed from given repositories"
msgstr ""

#: ../../client/solo/commands/query.py:171
msgid "list available packages"
msgstr "показать доступные пакеты"

#: ../../client/solo/commands/query.py:174
msgid "only list packages from given repositories"
msgstr ""

#: ../../client/solo/commands/query.py:182
msgid "show packages able to handle the given mimetype"
msgstr ""

#: ../../client/solo/commands/query.py:185
msgid "mimetype"
msgstr ""

#: ../../client/solo/commands/query.py:188
#: ../../client/solo/commands/query.py:203
msgid "only show installed packages"
msgstr ""

#: ../../client/solo/commands/query.py:197
msgid "associate file to packages able to handle it"
msgstr ""

#: ../../client/solo/commands/query.py:212
msgid "show runtime libraries needed by the given package"
msgstr ""

#: ../../client/solo/commands/query.py:222
msgid "search files not belonging to any packages"
msgstr ""

#: ../../client/solo/commands/query.py:228
msgid "show packages requiring the given library name"
msgstr ""

#: ../../client/solo/commands/query.py:231
msgid "library name (example: libdl.so.2)"
msgstr ""

#: ../../client/solo/commands/query.py:238
msgid "search package sets"
msgstr ""

#: ../../client/solo/commands/query.py:241
msgid "set name"
msgstr ""

#: ../../client/solo/commands/query.py:248
msgid "show packages owning the given slot"
msgstr ""

#: ../../client/solo/commands/query.py:251
msgid "slot name"
msgstr ""

#: ../../client/solo/commands/query.py:258
msgid "show packages owning the given tag"
msgstr ""

#: ../../client/solo/commands/query.py:261
msgid "tag name"
msgstr ""

#: ../../client/solo/commands/query.py:268
msgid "show packages at the given revision"
msgstr ""

#: ../../client/solo/commands/query.py:271
msgid "revision name"
msgstr ""

#: ../../client/solo/commands/query.py:278
msgid "show the direct dependencies graph for the given package"
msgstr ""

#: ../../client/solo/commands/query.py:285
#: ../../client/solo/commands/query.py:302
msgid ""
"include system packages, build-time dependencies and circular dependencies "
"information"
msgstr ""

#: ../../client/solo/commands/query.py:295
msgid "show the inverse dependencies graph for the given package"
msgstr ""

#: ../../client/solo/commands/query.py:312
msgid "show the number of updates available"
msgstr ""

#: ../../client/solo/commands/query.py:355
msgid "Belong Search"
msgstr "Поиск принадлежностей"

#: ../../client/solo/commands/query.py:407
#: ../../client/solo/commands/query.py:564
#: ../../client/solo/commands/query.py:658
#: ../../client/solo/commands/query.py:772
#: ../../client/solo/commands/query.py:962
#: ../../client/solo/commands/query.py:1331
#: ../../client/solo/commands/query.py:1390
#: ../../client/solo/commands/query.py:1453
#: ../../client/solo/commands/query.py:1501
msgid "Keyword"
msgstr "Ключевое слово"

#: ../../client/solo/commands/query.py:410
#: ../../client/solo/commands/query.py:579
#: ../../client/solo/commands/query.py:659
#: ../../client/solo/commands/query.py:704
#: ../../client/solo/commands/query.py:775
#: ../../client/solo/commands/query.py:964
#: ../../client/solo/commands/query.py:1029
#: ../../client/solo/commands/query.py:1278
#: ../../client/solo/commands/query.py:1332
#: ../../client/solo/commands/query.py:1393
#: ../../client/solo/commands/query.py:1456
#: ../../client/solo/commands/query.py:1504
#: ../../client/solo/commands/search.py:180
#: ../../client/solo/commands/match.py:221
msgid "Found"
msgstr "Найдено"

#: ../../client/solo/commands/query.py:412
#: ../../client/solo/commands/query.py:577
#: ../../client/solo/commands/query.py:661
#: ../../client/solo/commands/query.py:769
#: ../../client/solo/commands/query.py:959
#: ../../client/solo/commands/query.py:1330
#: ../../client/solo/commands/query.py:1388
#: ../../client/solo/commands/query.py:1451
#: ../../client/solo/commands/query.py:1499
#: ../../client/solo/commands/search.py:182
#: ../../client/solo/commands/match.py:223
msgid "entry"
msgid_plural "entries"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../client/solo/commands/query.py:429
msgid "ChangeLog Search"
msgstr "Поиск логов изменений"

#: ../../client/solo/commands/query.py:439
#: ../../client/solo/commands/query.py:447
#: ../../client/solo/commands/_manage.py:385
msgid "No match for"
msgstr "Ничего не найдено для"

#: ../../client/solo/commands/query.py:471
msgid "No ChangeLog available"
msgstr "Недоступно ни одного лога изменений"

#: ../../client/solo/commands/query.py:486
msgid "Attention: developer-repo option not enabled"
msgstr ""

#: ../../client/solo/commands/query.py:506
msgid "Reverse Dependencies Search"
msgstr ""

#: ../../client/solo/commands/query.py:566
#: ../../client/solo/commands/libtest.py:198
msgid "Matched"
msgstr "Подсчитано"

#: ../../client/solo/commands/query.py:572
#: ../../client/solo/commands/repo.py:120
msgid "from repository"
msgstr "из репозитория"

#: ../../client/solo/commands/query.py:574
msgid "from the installed packages repository"
msgstr ""

#: ../../client/solo/commands/query.py:602
msgid "Description Search"
msgstr "Поиск описания"

#: ../../client/solo/commands/query.py:626
#: ../../client/solo/commands/query.py:782
#: ../../client/solo/commands/query.py:841
#: ../../client/solo/commands/query.py:973
#: ../../client/solo/commands/query.py:1338
#: ../../client/solo/commands/query.py:1401
#: ../../client/solo/commands/query.py:1464
#: ../../client/solo/commands/query.py:1512
msgid "No matches"
msgstr "Ничего не найдено"

#: ../../client/solo/commands/query.py:677
msgid "Files Search"
msgstr ""

#: ../../client/solo/commands/query.py:705
msgid "files"
msgstr "файлы"

#: ../../client/solo/commands/query.py:733
msgid "License Search"
msgstr "Поиск лицензии"

#: ../../client/solo/commands/query.py:799
msgid "Listing Packages"
msgstr ""

#: ../../client/solo/commands/query.py:892
msgid "Repository is not available"
msgstr ""

#: ../../client/solo/commands/query.py:923
msgid "Searching Mimetype"
msgstr ""

#: ../../client/solo/commands/query.py:1005
msgid "Needed Libraries Search"
msgstr ""

#: ../../client/solo/commands/query.py:1030
msgid "libraries"
msgstr "библиотеки"

#: ../../client/solo/commands/query.py:1046
msgid "Orphans Search"
msgstr "Поиск осиротевших файлов"

#: ../../client/solo/commands/query.py:1114
msgid "Analyzing"
msgstr "Анализ"

#: ../../client/solo/commands/query.py:1128
msgid "Analyzed directories"
msgstr "Проанализированные директории"

#: ../../client/solo/commands/query.py:1133
msgid "Masked directories"
msgstr "Замаскированные директории"

#: ../../client/solo/commands/query.py:1138
msgid "Number of files collected on the filesystem"
msgstr "Число файлов, собранных в файловой системе"

#: ../../client/solo/commands/query.py:1143
msgid "Now searching among installed packages"
msgstr ""

#: ../../client/solo/commands/query.py:1174
msgid "Checking"
msgstr ""

#: ../../client/solo/commands/query.py:1189
msgid "Number of total files"
msgstr "Всего файлов"

#: ../../client/solo/commands/query.py:1195
msgid "Number of matching files"
msgstr "Число подсчитанных файлов"

#: ../../client/solo/commands/query.py:1202
msgid "Number of orphaned files"
msgstr "Число осиротевших файлов"

#: ../../client/solo/commands/query.py:1208
msgid "Writing file to disk"
msgstr "Запись файла на диск"

#: ../../client/solo/commands/query.py:1238
msgid "Total space wasted"
msgstr ""

#: ../../client/solo/commands/query.py:1256
msgid "Required Packages Search"
msgstr ""

#: ../../client/solo/commands/query.py:1274
msgid "Library"
msgstr "Библиотека"

#: ../../client/solo/commands/query.py:1298
msgid "Package Set Search"
msgstr "Поиск набора пакетов"

#: ../../client/solo/commands/query.py:1354
msgid "Slot Search"
msgstr "Поиск слота"

#: ../../client/solo/commands/query.py:1417
msgid "Tag Search"
msgstr "Поиск тэга"

#: ../../client/solo/commands/query.py:1480
msgid "Revision Search"
msgstr ""

#: ../../client/solo/commands/query.py:1549
#: ../../magneto/src/magneto/gtk/magneto.glade.h:2
msgid "Available Updates"
msgstr "Доступные обновления"

#: ../../client/solo/commands/query.py:1564
msgid "Packages to update:"
msgstr ""

#: ../../client/solo/commands/query.py:1567
msgid "Packages to remove:"
msgstr ""

#: ../../client/solo/commands/query.py:1571
msgid "Packages already up-to-date:"
msgstr ""

#: ../../client/solo/commands/query.py:1574
msgid "Critical updates found:"
msgstr ""

#: ../../client/solo/commands/config.py:69
#: ../../client/solo/commands/security.py:93
#: ../../client/solo/commands/security.py:165
#: ../../client/solo/commands/remove.py:74
#: ../../client/solo/commands/source.py:69
#: ../../client/solo/commands/install.py:77
#: ../../client/solo/commands/upgrade.py:72
#: ../../client/solo/commands/rescue.py:86
#: ../../client/solo/commands/download.py:69
msgid "show what would be done"
msgstr ""

#: ../../client/solo/commands/config.py:76
#: ../../client/solo/commands/remove.py:81
#: ../../client/solo/commands/search.py:76
#: ../../client/solo/commands/command.py:120
#: ../../client/solo/commands/source.py:76
#: ../../client/solo/commands/match.py:90
#: ../../client/solo/commands/install.py:84
#: ../../client/solo/commands/upgrade.py:79
#: ../../client/solo/commands/download.py:76
msgid "verbose output"
msgstr ""

#: ../../client/solo/commands/config.py:126
#: ../../client/solo/commands/status.py:130
#: ../../client/solo/commands/status.py:137
#: ../../client/solo/commands/status.py:144
#: ../../client/solo/commands/remove.py:220
#: ../../client/solo/commands/remove.py:284
#: ../../client/solo/commands/_manage.py:152
#: ../../client/solo/commands/_manage.py:163
#: ../../client/solo/commands/install.py:270
#: ../../client/solo/commands/install.py:296
#: ../../client/solo/commands/install.py:382
msgid "Not available"
msgstr "Недоступно"

#: ../../client/solo/commands/config.py:130
msgid "installed from"
msgstr ""

#: ../../client/solo/commands/config.py:141
#: ../../client/solo/commands/remove.py:307
#: ../../client/solo/commands/_manage.py:229
msgid "Packages involved"
msgstr "Включённые пакеты"

#: ../../client/solo/commands/config.py:165
msgid "configure"
msgstr ""

#: ../../client/solo/commands/config.py:187
msgid "configure installed packages"
msgstr ""

#: ../../client/solo/commands/status.py:105
msgid "Repository name"
msgstr "Имя репозитория"

#: ../../client/solo/commands/status.py:114
msgid "Revision"
msgstr ""

#: ../../client/solo/commands/status.py:126
msgid "Repository class"
msgstr ""

#: ../../client/solo/commands/status.py:140
msgid "Notice Board"
msgstr ""

#: ../../client/solo/commands/status.py:147
msgid "Path"
msgstr ""

#: ../../client/solo/commands/status.py:160
msgid "Package URLs"
msgstr ""

#: ../../client/solo/commands/status.py:175
msgid "show Repositories status"
msgstr ""

#: ../../client/solo/commands/update.py:64
#: ../../client/solo/commands/ugc.py:236
#: ../../client/solo/commands/notice.py:57
msgid "repository"
msgstr "репозиторий"

#: ../../client/solo/commands/update.py:67
msgid "force update"
msgstr ""

#: ../../client/solo/commands/update.py:103
msgid "You must be either root or in this group:"
msgstr ""

#: ../../client/solo/commands/update.py:123
msgid "Sending the update request to Entropy Services"
msgstr ""

#: ../../client/solo/commands/update.py:124
msgid "Repositories will be updated in background"
msgstr ""

#: ../../client/solo/commands/update.py:133
msgid "app-admin/rigo-daemon not installed. Update not allowed."
msgstr ""

#: ../../client/solo/commands/update.py:200
msgid "Have a nice day"
msgstr ""

#: ../../client/solo/commands/update.py:205
msgid "Repositories update not allowed"
msgstr ""

#: ../../client/solo/commands/update.py:230
#: ../../client/solo/commands/hop.py:116
msgid "No repositories specified in"
msgstr "Ни одного репозитория не отмечено в"

#: ../../client/solo/commands/update.py:269
#: ../../client/solo/commands/notice.py:123
msgid "Notice board not available"
msgstr "Доска уведомлений недоступна"

#: ../../client/solo/commands/update.py:283
msgid "Notice board"
msgstr "Доска уведомлений"

#. title
#: ../../client/solo/commands/update.py:296
#: ../../client/solo/commands/security.py:219
#: ../../client/solo/commands/ugc.py:357 ../../client/solo/commands/ugc.py:616
#: ../../client/solo/commands/notice.py:146
#: ../../client/solo/commands/notice.py:183
msgid "Title"
msgstr "Заголовок"

#: ../../client/solo/commands/update.py:305
msgid "update repositories"
msgstr ""

#: ../../client/solo/commands/security.py:68
#: ../../client/solo/commands/security.py:631
msgid "system security tools"
msgstr ""

#: ../../client/solo/commands/security.py:73
msgid "verify installed files using stored checksums"
msgstr ""

#: ../../client/solo/commands/security.py:77
msgid "consider mtime instead of SHA256 (false positives ahead)"
msgstr ""

#: ../../client/solo/commands/security.py:81
msgid "update hashes and mtime (useful after editing config files)"
msgstr ""

#: ../../client/solo/commands/security.py:85
msgid "reinstall faulty packages"
msgstr ""

#: ../../client/solo/commands/security.py:96
#: ../../client/solo/commands/security.py:168
#: ../../client/solo/commands/install.py:98
#: ../../client/solo/commands/upgrade.py:93
msgid "just download packages"
msgstr "просто загрузить пакеты"

#: ../../client/solo/commands/security.py:112
msgid "download the latest Security Advisories"
msgstr "загружает последние уведомления о безопасности"

#: ../../client/solo/commands/security.py:116
msgid "force download"
msgstr ""

#: ../../client/solo/commands/security.py:125
msgid "list all the available Security Advisories"
msgstr "выдаёт список всех доступных уведомлений о безопасности"

#: ../../client/solo/commands/security.py:131
msgid "list only affected"
msgstr "показать только список уязвимого"

#: ../../client/solo/commands/security.py:134
msgid "list only unaffected"
msgstr "показать только список неуязвимого"

#: ../../client/solo/commands/security.py:143
msgid "show information about provided advisories identifiers"
msgstr "показать информацию о идентификаторах предоставленных уведомлений"

#: ../../client/solo/commands/security.py:148
msgid "advisory indentifier"
msgstr ""

#: ../../client/solo/commands/security.py:155
msgid "automatically install all the available security updates"
msgstr "автоматически установить все доступные обновления безопасности"

#: ../../client/solo/commands/security.py:215
msgid "Advisory Identifier"
msgstr ""

#: ../../client/solo/commands/security.py:247
msgid "Background"
msgstr "Фон"

#: ../../client/solo/commands/security.py:254
msgid "Exploitable"
msgstr "Добываемое"

#: ../../client/solo/commands/security.py:260
msgid "Impact"
msgstr "Влияние"

#: ../../client/solo/commands/security.py:267
msgid "Impact type"
msgstr "Тип влияния"

#: ../../client/solo/commands/security.py:272
msgid "Revised"
msgstr "Проверено"

#: ../../client/solo/commands/security.py:277
msgid "Announced"
msgstr "Объявлено"

#: ../../client/solo/commands/security.py:282
msgid "Synopsis"
msgstr "Перспектива"

#: ../../client/solo/commands/security.py:289
msgid "References"
msgstr "Ссылки"

#: ../../client/solo/commands/security.py:295
msgid "Upstream bugs"
msgstr "Баги в процессе"

#: ../../client/solo/commands/security.py:301
msgid "Affected"
msgstr "Уязвимо"

#: ../../client/solo/commands/security.py:309
msgid "vulnerable versions"
msgstr "уязвимые версии"

#: ../../client/solo/commands/security.py:312
msgid "unaffected versions"
msgstr "неуязвимые версии"

#: ../../client/solo/commands/security.py:317
msgid "Workaround"
msgstr "Оптимизация"

#: ../../client/solo/commands/security.py:324
msgid "Resolution"
msgstr "Разрешение"

#: ../../client/solo/commands/security.py:351
msgid "Checking system files"
msgstr ""

#: ../../client/solo/commands/security.py:379
msgid "no checksum information"
msgstr ""

#: ../../client/solo/commands/security.py:428
msgid "found altered files"
msgstr ""

#: ../../client/solo/commands/security.py:447
msgid "assimilated new hashes and mtime"
msgstr ""

#: ../../client/solo/commands/security.py:458
msgid "unavailable"
msgstr ""

#: ../../client/solo/commands/security.py:465
msgid "No altered files found"
msgstr ""

#: ../../client/solo/commands/security.py:491
msgid "Altered files have been found"
msgstr ""

#: ../../client/solo/commands/security.py:495
msgid "Packages have been reinstalled successfully"
msgstr ""

#: ../../client/solo/commands/security.py:527
msgid "No advisories available or applicable"
msgstr "Нет доступных либо применимых уведомлений"

#: ../../client/solo/commands/security.py:570
msgid "Advisory does not exist"
msgstr ""

#: ../../client/solo/commands/security.py:596
msgid "Calculating security updates"
msgstr "Подсчёт обновлений безопасности"

#: ../../client/solo/commands/security.py:615
msgid "All the available updates have been already installed"
msgstr "Все возможные обновления уже были установлены"

#: ../../client/solo/commands/moo.py:114
msgid "moo at user"
msgstr ""

#: ../../client/solo/commands/moo.py:121
msgid "bow to lxnay"
msgstr ""

#: ../../client/solo/commands/yell.py:34
msgid "You should run equo --help"
msgstr "Вы должны запустить equo --help"

#: ../../client/solo/commands/yell.py:35
msgid "You didn't run equo --help, did you?"
msgstr "Вы не запустили equo --help, так ведь?"

#: ../../client/solo/commands/yell.py:36
msgid "Did you even read equo --help??"
msgstr "Вы когда-нибудь читали equo --help??"

#: ../../client/solo/commands/yell.py:37
msgid "I give up. Run that equo --help !!!!!!!"
msgstr "Я умываю руки. Запусти уже этот чёртов equo --help !!!!!!!"

#: ../../client/solo/commands/yell.py:38
msgid "OH MY GOD. RUN equo --heeeeeeeeeeeeeelp"
msgstr "О, Боже. ЗАПУСТИ ЭТОТ ГРЁБАНЫЙ equo --help..."

#: ../../client/solo/commands/yell.py:39
msgid "Illiteracy is a huge problem in this world"
msgstr "Невежество - путь к ереси."

#: ../../client/solo/commands/yell.py:40
msgid "Ok i give up, you are hopeless"
msgstr "Окей, я сдаюсь. Ты неисправим."

#: ../../client/solo/commands/yell.py:41 ../../client/solo/commands/yell.py:42
#: ../../client/solo/commands/yell.py:43 ../../client/solo/commands/yell.py:44
#: ../../client/solo/commands/yell.py:45 ../../client/solo/commands/yell.py:47
msgid "Go to hell."
msgstr "Иди к чёрту."

#: ../../client/solo/commands/yell.py:46
msgid "Stop that, you idiot."
msgstr ""

#: ../../client/solo/commands/yell.py:117
msgid "yell at user"
msgstr ""

#: ../../client/solo/commands/remove.py:88
#: ../../client/solo/commands/source.py:90
#: ../../client/solo/commands/install.py:110
#: ../../client/solo/commands/upgrade.py:105
#: ../../client/solo/commands/download.py:89
msgid "exclude package dependencies"
msgstr ""

#: ../../client/solo/commands/remove.py:94
#: ../../client/solo/commands/source.py:103
#: ../../client/solo/commands/install.py:123
#: ../../client/solo/commands/upgrade.py:111
#: ../../client/solo/commands/download.py:102
msgid "do not calculate dependencies recursively"
msgstr ""

#: ../../client/solo/commands/remove.py:100
#: ../../client/solo/commands/source.py:109
#: ../../client/solo/commands/install.py:129
#: ../../client/solo/commands/upgrade.py:117
#: ../../client/solo/commands/download.py:108
msgid "include dependencies no longer needed"
msgstr ""

#: ../../client/solo/commands/remove.py:106
#: ../../client/solo/commands/install.py:135
#: ../../client/solo/commands/upgrade.py:123
msgid "when used with --deep, include virtual packages"
msgstr ""

#: ../../client/solo/commands/remove.py:112
#: ../../client/solo/commands/install.py:141
#: ../../client/solo/commands/upgrade.py:136
msgid "remove package configuration files no longer needed"
msgstr ""

#: ../../client/solo/commands/remove.py:118
msgid "force system packages removal (dangerous!)"
msgstr ""

#: ../../client/solo/commands/remove.py:173
msgid "removal"
msgstr ""

#: ../../client/solo/commands/remove.py:189
#: ../../client/solo/commands/source.py:185
#: ../../client/solo/commands/download.py:185
msgid "All done"
msgstr "Всё сделано"

#: ../../client/solo/commands/remove.py:201
msgid "These are the packages that should be MANUALLY removed"
msgstr ""

#: ../../client/solo/commands/remove.py:207
#: ../../client/solo/commands/install.py:362
msgid "These are the packages that would be removed"
msgstr "Здесь перечислены пакеты, которые должны быть удалены"

#: ../../client/solo/commands/remove.py:257
msgid "These are the chosen packages"
msgstr "Вот выбранные пакеты"

#: ../../client/solo/commands/remove.py:270
msgid "vital package"
msgstr "жизненно важный пакет"

#: ../../client/solo/commands/remove.py:271
msgid "Removal forbidden"
msgstr "Удаление запрещено"

#: ../../client/solo/commands/remove.py:321
msgid "Packages that would be removed"
msgstr ""

#: ../../client/solo/commands/remove.py:351
#: ../../client/solo/commands/install.py:453
msgid "Freed disk space"
msgstr "Свободное место на диске"

#: ../../client/solo/commands/remove.py:358
msgid "Total bandwidth wasted"
msgstr ""

#: ../../client/solo/commands/remove.py:388
#: ../../client/solo/commands/source.py:179
#: ../../client/solo/commands/install.py:543
#: ../../client/solo/commands/rescue.py:641
#: ../../client/solo/commands/download.py:179
msgid "Nothing to do"
msgstr "Делать нечего"

#: ../../client/solo/commands/remove.py:395
msgid "Would you like to calculate dependencies ?"
msgstr "Желаете ли подсчитать зависимости?"

#: ../../client/solo/commands/remove.py:402
msgid "Would you like to remove them now ?"
msgstr "Вы желаете удалить их сейчас ?"

#: ../../client/solo/commands/remove.py:423
msgid "Ouch!"
msgstr ""

#: ../../client/solo/commands/remove.py:424
msgid "the following system packages were pulled in"
msgstr ""

#: ../../client/solo/commands/remove.py:447
msgid "Would you like to proceed ?"
msgstr "Желаете продолжить ?"

#: ../../client/solo/commands/remove.py:462
msgid "remove packages from system"
msgstr ""

#: ../../client/solo/commands/search.py:68
msgid "search keyword"
msgstr ""

#: ../../client/solo/commands/search.py:72
#: ../../client/solo/commands/repo.py:133
#: ../../client/solo/commands/match.py:86
msgid "quiet output, for scripting purposes"
msgstr ""

#: ../../client/solo/commands/search.py:81
msgid "search among installed packages only"
msgstr ""

#: ../../client/solo/commands/search.py:85
msgid "search among available packages only"
msgstr ""

#: ../../client/solo/commands/search.py:167
msgid "Searching"
msgstr "Поиск"

#: ../../client/solo/commands/search.py:214
msgid "search packages in repositories"
msgstr "найти пакеты в репозиториях"

#: ../../client/solo/commands/deptest.py:64
#: ../../client/solo/commands/libtest.py:67
#: ../../client/solo/commands/unused.py:65
msgid "show less details (useful for scripting)"
msgstr "показать меньше деталей (полезно для скриптинга)"

#: ../../client/solo/commands/deptest.py:67
#: ../../client/solo/commands/libtest.py:74 ../kernel-switcher:44
msgid "just show what would be done"
msgstr "показывает только то, что будет сделано"

#: ../../client/solo/commands/deptest.py:104
msgid "Running dependency test"
msgstr "Запуск теста зависимостей"

#: ../../client/solo/commands/deptest.py:159
msgid "These are the dependencies not found"
msgstr "Имеются ненайденные зависимости"

#: ../../client/solo/commands/deptest.py:169
msgid "Needed by"
msgstr "Необходимо для"

#: ../../client/solo/commands/deptest.py:180
msgid "Would you like to install the packages ?"
msgstr ""

#: ../../client/solo/commands/deptest.py:187
#: ../../client/solo/commands/libtest.py:227
msgid "Installing available packages in"
msgstr "Установка доступных пакетов в"

#: ../../client/solo/commands/deptest.py:188
#: ../../client/solo/commands/libtest.py:228
msgid "10 seconds"
msgstr "10 секунд"

#: ../../client/solo/commands/deptest.py:206
msgid "look for unsatisfied dependencies"
msgstr "ищет неудовлетворенные зависимости"

#: ../../client/solo/commands/command.py:96
msgid "not a valid directory"
msgstr ""

#: ../../client/solo/commands/command.py:123
#: ../../client/solo/commands/source.py:83
#: ../../client/solo/commands/install.py:91
#: ../../client/solo/commands/upgrade.py:86
#: ../../client/solo/commands/download.py:83
msgid "quiet output"
msgstr ""

#: ../../client/solo/commands/command.py:349
msgid "Acquiring Entropy Resources Lock, please wait..."
msgstr ""

#: ../../client/solo/commands/command.py:356
#: ../../client/solo/commands/command.py:386 ../kernel-switcher:57
msgid "Another Entropy is currently running."
msgstr ""

#: ../../client/solo/commands/command.py:414
msgid "When you wrote"
msgstr "Как Вы написали"

#: ../../client/solo/commands/command.py:416
msgid "You Meant(tm)"
msgstr "А ты думал (TM)"

#: ../../client/solo/commands/command.py:417
msgid "one of these below?"
msgstr "один из нижеуказанных?"

#: ../../client/solo/commands/cache.py:60
msgid "manage cache"
msgstr ""

#: ../../client/solo/commands/cache.py:64
msgid "clean Entropy Library Cache"
msgstr ""

#: ../../client/solo/commands/cache.py:67
msgid "show more details"
msgstr "показать больше деталей"

#: ../../client/solo/commands/cache.py:70
msgid "print results in a scriptable way"
msgstr "отображает результаты через скрипты"

#: ../../client/solo/commands/cache.py:122
msgid "Cleaning Entropy cache, please wait ..."
msgstr ""

#: ../../client/solo/commands/cache.py:129
msgid "Entropy cache cleaned."
msgstr ""

#: ../../client/solo/commands/cache.py:140
msgid "manage Entropy Library Cache"
msgstr ""

#: ../../client/solo/commands/ugc.py:71 ../../client/solo/commands/ugc.py:1007
msgid "manage User Generated Content"
msgstr ""

#: ../../client/solo/commands/ugc.py:75
msgid "login against the given repository"
msgstr ""

#: ../../client/solo/commands/ugc.py:77 ../../client/solo/commands/ugc.py:87
#: ../../client/solo/commands/ugc.py:99 ../../client/solo/commands/ugc.py:140
#: ../../client/solo/commands/repo.py:74 ../../client/solo/commands/repo.py:82
#: ../../client/solo/commands/repo.py:89
#: ../../client/solo/commands/repo.py:112
#: ../../client/solo/commands/repo.py:140
msgid "repository name"
msgstr ""

#: ../../client/solo/commands/ugc.py:80 ../../client/solo/commands/ugc.py:90
msgid "force action"
msgstr "принудительно запустить действие"

#: ../../client/solo/commands/ugc.py:85
msgid "logout from the given repository"
msgstr ""

#: ../../client/solo/commands/ugc.py:96
msgid ""
"manage package documents in the selected repository (comments, files, "
"videos)"
msgstr ""

#: ../../client/solo/commands/ugc.py:104
msgid "manage Documents"
msgstr ""

#: ../../client/solo/commands/ugc.py:108
msgid "get available documents for the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:111 ../../client/solo/commands/ugc.py:119
#: ../../client/solo/commands/ugc.py:127 ../../client/solo/commands/ugc.py:151
#: ../../client/solo/commands/ugc.py:158
msgid "package name (example: x11-libs/qt)"
msgstr ""

#: ../../client/solo/commands/ugc.py:116
msgid "add a new document to the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:124
msgid "remove documents from database using their identifiers"
msgstr "удалить документы из базы данных, используя их наименования"

#: ../../client/solo/commands/ugc.py:130
msgid "document identifier"
msgstr ""

#: ../../client/solo/commands/ugc.py:137
msgid "manage package votes in the selected repository"
msgstr ""

#: ../../client/solo/commands/ugc.py:145
msgid "manage Votes"
msgstr ""

#: ../../client/solo/commands/ugc.py:149
msgid "get vote for the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:156
msgid "add vote to the provided package name"
msgstr ""

#: ../../client/solo/commands/ugc.py:196 ../../client/solo/commands/ugc.py:300
msgid "Invalid repository"
msgstr "Некорректный репозиторий"

#: ../../client/solo/commands/ugc.py:207 ../../client/solo/commands/ugc.py:311
#: ../../client/solo/commands/ugc.py:435 ../../client/solo/commands/ugc.py:517
#: ../../client/solo/commands/ugc.py:733 ../../client/solo/commands/ugc.py:850
#: ../../client/solo/commands/ugc.py:888
msgid "Repository does not support Entropy Services."
msgstr ""

#: ../../client/solo/commands/ugc.py:217
msgid "Already logged in as"
msgstr "Уже вошли в систему как"

#: ../../client/solo/commands/ugc.py:219
msgid "Please logout first"
msgstr "Пожалуйста, сначала выйдите из системы."

#: ../../client/solo/commands/ugc.py:231
msgid "Username"
msgstr "Имя пользователя"

#: ../../client/solo/commands/ugc.py:232
msgid "Password"
msgstr "Пароль"

#: ../../client/solo/commands/ugc.py:236
msgid "Please login against"
msgstr "Пожалуйста, залогиньтесь снова"

#: ../../client/solo/commands/ugc.py:244
msgid "Login aborted. Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:258
msgid "Authentication error. Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:267
msgid "Communication error. Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:277
msgid "Successfully logged in as"
msgstr "Успешно вошли в систему как"

#: ../../client/solo/commands/ugc.py:284
msgid "From now on, any UGC action will be committed as this user"
msgstr "С текущего момента любое действие UGC будет определено как от этого пользователя"

#: ../../client/solo/commands/ugc.py:321
msgid "Not logged in."
msgstr ""

#: ../../client/solo/commands/ugc.py:330
msgid "User"
msgstr "Пользователь"

#: ../../client/solo/commands/ugc.py:332
msgid "has been logged out."
msgstr ""

#: ../../client/solo/commands/ugc.py:341
msgid "No title"
msgstr "Нет заголовка"

#: ../../client/solo/commands/ugc.py:368
#: ../../client/solo/commands/notice.py:189
msgid "Content"
msgstr "Содержимое"

#: ../../client/solo/commands/ugc.py:385
msgid "no votes"
msgstr ""

#: ../../client/solo/commands/ugc.py:391
msgid "current package vote"
msgstr ""

#: ../../client/solo/commands/ugc.py:452 ../../client/solo/commands/ugc.py:462
#: ../../client/solo/commands/ugc.py:479 ../../client/solo/commands/ugc.py:698
#: ../../client/solo/commands/ugc.py:769 ../../client/solo/commands/ugc.py:794
#: ../../client/solo/commands/ugc.py:806 ../../client/solo/commands/ugc.py:818
#: ../../client/solo/commands/ugc.py:863 ../../client/solo/commands/ugc.py:941
#: ../../client/solo/commands/ugc.py:976 ../../client/solo/commands/ugc.py:987
msgid "UGC error"
msgstr "Ошибка UGC"

#: ../../client/solo/commands/ugc.py:491
msgid "Number of downloads"
msgstr "Число загрузок"

#: ../../client/solo/commands/ugc.py:498
msgid "No User Generated Content available."
msgstr ""

#: ../../client/solo/commands/ugc.py:529 ../../client/solo/commands/ugc.py:900
msgid "Not logged in, please login first."
msgstr ""

#: ../../client/solo/commands/ugc.py:539
msgid "Add document"
msgstr "Добавить документ"

#: ../../client/solo/commands/ugc.py:544
msgid "text comment"
msgstr "текстовый документ"

#: ../../client/solo/commands/ugc.py:545
msgid "icon"
msgstr ""

#: ../../client/solo/commands/ugc.py:546
msgid "simple file"
msgstr "простой файл"

#: ../../client/solo/commands/ugc.py:547
msgid "simple image"
msgstr "простое изображение"

#: ../../client/solo/commands/ugc.py:548
msgid "video"
msgstr ""

#: ../../client/solo/commands/ugc.py:565
msgid "Insert document title"
msgstr "Введите заголовок документа"

#: ../../client/solo/commands/ugc.py:567
msgid "Insert document description/comment"
msgstr "Введите описание/комментарий к документу"

#: ../../client/solo/commands/ugc.py:570
msgid "Insert document's keywords (space separated)"
msgstr "Введите ключевые слова документа (разделяя их пробелом)"

#: ../../client/solo/commands/ugc.py:572
msgid "Choose document type"
msgstr "Выберите тип документа"

#: ../../client/solo/commands/ugc.py:577 ../../client/solo/commands/ugc.py:597
msgid "Entropy UGC document submission"
msgstr "Представление документов Entropy UGC"

#: ../../client/solo/commands/ugc.py:594
msgid "Insert document path"
msgstr "Введите путь к документу"

#: ../../client/solo/commands/ugc.py:610 ../../client/solo/commands/ugc.py:750
#: ../../client/solo/commands/ugc.py:953
msgid "Please review your submission"
msgstr "Пожалуйста, оставьте своё мнение"

#: ../../client/solo/commands/ugc.py:638
msgid "Document path"
msgstr "Путь к документу"

#: ../../client/solo/commands/ugc.py:645
msgid "Document type"
msgstr "Тип документа"

#: ../../client/solo/commands/ugc.py:651 ../../client/solo/commands/ugc.py:965
msgid "Do you want to submit?"
msgstr ""

#: ../../client/solo/commands/ugc.py:683
msgid "Invalid document"
msgstr ""

#: ../../client/solo/commands/ugc.py:712
msgid "Document added, thank you!"
msgstr "Документ добавлен, спасибо!"

#: ../../client/solo/commands/ugc.py:743
msgid "Documents removal"
msgstr "Удаление документов"

#: ../../client/solo/commands/ugc.py:756
msgid "Document identifiers"
msgstr "Имена документов"

#: ../../client/solo/commands/ugc.py:761
msgid "Would you like to review them?"
msgstr ""

#: ../../client/solo/commands/ugc.py:784
msgid "Would you like to continue with the removal?"
msgstr ""

#: ../../client/solo/commands/ugc.py:807
msgid "cannot get the requested Document"
msgstr ""

#: ../../client/solo/commands/ugc.py:827
msgid "UGC status"
msgstr "Статус UGC"

#: ../../client/solo/commands/ugc.py:828
msgid "removed successfully"
msgstr ""

#: ../../client/solo/commands/ugc.py:909
msgid "add vote"
msgstr ""

#: ../../client/solo/commands/ugc.py:917
msgid "Insert your vote (from 1 to 5)"
msgstr "Введите Ваш голос (от 1 до 5)"

#: ../../client/solo/commands/ugc.py:921
msgid "Entropy UGC vote submission"
msgstr "Подчинение оценки Entropy UGC"

#: ../../client/solo/commands/ugc.py:942
msgid "invalid vote, must be in range"
msgstr ""

#: ../../client/solo/commands/ugc.py:959
msgid "Vote"
msgstr "Проголосовать"

#: ../../client/solo/commands/ugc.py:988
msgid "already voted"
msgstr ""

#: ../../client/solo/commands/ugc.py:996
msgid "vote added, thank you!"
msgstr ""

#: ../../client/solo/commands/help.py:70
msgid "Entropy Command Line Client, Equo"
msgstr ""

#: ../../client/solo/commands/help.py:77
msgid "force colored output"
msgstr ""

#: ../../client/solo/commands/help.py:102 ../kernel-switcher:155
msgid "this help"
msgstr ""

#: ../../client/solo/commands/repo.py:67
#: ../../client/solo/commands/repo.py:733
msgid "manage repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:71
msgid "enable repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:79
msgid "disable repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:87
msgid "add a repository"
msgstr ""

#: ../../client/solo/commands/repo.py:92
msgid "repository description"
msgstr ""

#: ../../client/solo/commands/repo.py:96
msgid "repository database URL"
msgstr ""

#: ../../client/solo/commands/repo.py:100
msgid "repository packages URL"
msgstr ""

#: ../../client/solo/commands/repo.py:104
msgid "repository compression format"
msgstr ""

#: ../../client/solo/commands/repo.py:109
msgid "remove a repository"
msgstr ""

#: ../../client/solo/commands/repo.py:117
msgid "rename a repository"
msgstr ""

#: ../../client/solo/commands/repo.py:123
msgid "to repository"
msgstr ""

#: ../../client/solo/commands/repo.py:128
msgid "list active repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:137
msgid "reorder mirrors basing on response time"
msgstr ""

#: ../../client/solo/commands/repo.py:143
msgid "simulate execution"
msgstr ""

#: ../../client/solo/commands/repo.py:148
msgid "merge content of source repository to destination"
msgstr ""

#: ../../client/solo/commands/repo.py:150
msgid "source repository"
msgstr ""

#: ../../client/solo/commands/repo.py:152
msgid "destination repository"
msgstr ""

#: ../../client/solo/commands/repo.py:156
msgid "also remove dependency conflicts during merge"
msgstr ""

#: ../../client/solo/commands/repo.py:248
msgid "repository already enabled"
msgstr ""

#: ../../client/solo/commands/repo.py:257
#: ../../client/solo/commands/repo.py:311
#: ../../client/solo/commands/repo.py:596
msgid "repository not available"
msgstr ""

#: ../../client/solo/commands/repo.py:277
msgid "repository enabled"
msgstr ""

#: ../../client/solo/commands/repo.py:283
msgid "cannot enable repository"
msgstr ""

#: ../../client/solo/commands/repo.py:302
msgid "repository already disabled"
msgstr ""

#: ../../client/solo/commands/repo.py:333
#: ../../client/solo/commands/repo.py:347
msgid "cannot disable repository"
msgstr ""

#: ../../client/solo/commands/repo.py:341
msgid "repository disabled"
msgstr ""

#: ../../client/solo/commands/repo.py:369
msgid "Repository id:"
msgstr ""

#: ../../client/solo/commands/repo.py:372
msgid "Description:"
msgstr ""

#: ../../client/solo/commands/repo.py:375
msgid "Repository format:"
msgstr ""

#: ../../client/solo/commands/repo.py:379
msgid "Packages URL:"
msgstr ""

#: ../../client/solo/commands/repo.py:381
msgid "Repository URL:"
msgstr ""

#: ../../client/solo/commands/repo.py:403
#: ../../client/solo/commands/repo.py:529
msgid "repository added succesfully"
msgstr ""

#: ../../client/solo/commands/repo.py:408
#: ../../client/solo/commands/repo.py:539
msgid "cannot add repository"
msgstr ""

#: ../../client/solo/commands/repo.py:430
#: ../../client/solo/commands/repo.py:479
msgid "repository id not available"
msgstr ""

#: ../../client/solo/commands/repo.py:450
msgid "repository removed succesfully"
msgstr ""

#: ../../client/solo/commands/repo.py:456
msgid "cannot remove repository"
msgstr ""

#: ../../client/solo/commands/repo.py:487
msgid "repository id already available"
msgstr ""

#: ../../client/solo/commands/repo.py:495
msgid "repository id is invalid"
msgstr ""

#: ../../client/solo/commands/repo.py:610
msgid "mirror order:"
msgstr ""

#: ../../client/solo/commands/repo.py:620
msgid "mirrors sorted successfully"
msgstr ""

#: ../../client/solo/commands/repo.py:639
#: ../../client/solo/commands/repo.py:646
msgid "repository not found"
msgstr ""

#: ../../client/solo/commands/repo.py:655
msgid "repository cannot be source and destination"
msgstr ""

#: ../../client/solo/commands/repo.py:663
msgid "merging repositories"
msgstr ""

#: ../../client/solo/commands/repo.py:679
msgid "working on repository"
msgstr ""

#: ../../client/solo/commands/repo.py:696
msgid "merging package"
msgstr ""

#: ../../client/solo/commands/repo.py:720
msgid "done merging packages"
msgstr ""

#: ../../client/solo/commands/libtest.py:80
msgid "print broken files to stdout"
msgstr "записывает испорченные файлы в stdout"

#: ../../client/solo/commands/libtest.py:85
msgid "dump results to files"
msgstr "записать результаты в файлы"

#: ../../client/solo/commands/libtest.py:140
#: ../../client/solo/commands/libtest.py:173
msgid "System is healthy"
msgstr "В системе все в порядке"

#: ../../client/solo/commands/libtest.py:179
msgid "Libraries/Executables statistics"
msgstr "Статистика библиотек/исполняемых файлов"

#: ../../client/solo/commands/libtest.py:185
msgid "Not matched"
msgstr "Не подсчитано"

#: ../../client/solo/commands/libtest.py:221
msgid "Would you like to install them ?"
msgstr "Вы желаете их инсталлировать ?"

#: ../../client/solo/commands/libtest.py:249
msgid "look for missing libraries"
msgstr "ищет испорченные библиотеки"

#: ../../client/solo/commands/unused.py:71
msgid "sort packages by size"
msgstr ""

#: ../../client/solo/commands/unused.py:76
msgid "include packages installed by user"
msgstr ""

#: ../../client/solo/commands/unused.py:115
msgid ""
"Running unused packages test, pay attention, there can be false positives"
msgstr ""

#: ../../client/solo/commands/unused.py:163
msgid "look for unused packages (pay attention)"
msgstr "ищет неиспользуемые пакеты (обратите внимание)"

#: ../../client/solo/commands/_manage.py:89
msgid "Scanning configuration files to update"
msgstr ""

#: ../../client/solo/commands/_manage.py:98
msgid "No configuration files to update."
msgstr ""

#: ../../client/solo/commands/_manage.py:103
#, python-format
msgid "There is %s configuration file needing update"
msgid_plural "There are %s configuration files needing update"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../client/solo/commands/_manage.py:111
msgid "Please run"
msgstr "Будьте добры запустить"

#: ../../client/solo/commands/_manage.py:127
msgid "These are the selected packages"
msgstr ""

#: ../../client/solo/commands/_manage.py:180
msgid "Versions"
msgstr "Версии"

#: ../../client/solo/commands/_manage.py:204
#: ../../client/solo/commands/_manage.py:210
msgid "Reinstall"
msgstr "Переустановить"

#: ../../client/solo/commands/_manage.py:205
msgid "Switch repo"
msgstr "Переключить репозиторий"

#: ../../client/solo/commands/_manage.py:212
msgid "Install"
msgstr "Инсталлировать"

#: ../../client/solo/commands/_manage.py:214
msgid "Upgrade"
msgstr "Обновить"

#: ../../client/solo/commands/_manage.py:216
msgid "Downgrade"
msgstr "Обновить до более старой версии"

#: ../../client/solo/commands/_manage.py:224
msgid "Action"
msgstr "Действие"

#: ../../client/solo/commands/_manage.py:236
msgid "Would you like to continue with the dependencies calculation?"
msgstr ""

#. every package matching app-foo is masked
#: ../../client/solo/commands/_manage.py:338
msgid "Every package matching"
msgstr "Подсчёт каждого пакета"

#: ../../client/solo/commands/_manage.py:340
msgid "is masked"
msgstr "замаскирован"

#: ../../client/solo/commands/_manage.py:358
msgid "Masking reason"
msgstr ""

#: ../../client/solo/commands/_manage.py:370
msgid "matching"
msgstr "подсчитывается"

#: ../../client/solo/commands/_manage.py:372
msgid "is broken"
msgstr "сломан"

#: ../../client/solo/commands/_manage.py:376
msgid "atom"
msgstr "атом"

#: ../../client/solo/commands/_manage.py:378
msgid "in"
msgstr "в"

#: ../../client/solo/commands/_manage.py:387
msgid "in repositories"
msgstr "в репозиториях"

#: ../../client/solo/commands/_manage.py:400
msgid "Not found"
msgstr "Не найдено"

#: ../../client/solo/commands/_manage.py:409
msgid "package needed by"
msgstr ""

#: ../../client/solo/commands/_manage.py:415
#: ../../client/solo/commands/_manage.py:784
#: ../../client/solo/commands/install.py:387
msgid "from"
msgstr "из"

#: ../../client/solo/commands/_manage.py:459
msgid "Skipped"
msgstr "Пропущено"

#: ../../client/solo/commands/_manage.py:480
msgid "is not installed"
msgstr "не установлено"

#: ../../client/solo/commands/_manage.py:510
msgid "Calculating dependencies"
msgstr "Подсчёт зависимостей"

#: ../../client/solo/commands/_manage.py:522
msgid "Dependencies not found"
msgstr ""

#: ../../client/solo/commands/_manage.py:533
msgid "Conflicting packages were pulled in"
msgstr ""

#: ../../client/solo/commands/_manage.py:549
msgid "Please mask conflicts using"
msgstr ""

#: ../../client/solo/commands/_manage.py:598
#: ../../client/solo/commands/_manage.py:639
msgid "download"
msgstr ""

#: ../../client/solo/commands/_manage.py:664
msgid "Repositories are old, please run:"
msgstr ""

#: ../../client/solo/commands/_manage.py:688
msgid "Please update the following critical packages"
msgstr "Пожалуйста, обновитье следующие критические пакеты"

#: ../../client/solo/commands/_manage.py:699
msgid "You should install them as soon as possible"
msgstr "Вы должны установить их как можно раньше"

#: ../../client/solo/commands/_manage.py:720
msgid "Read the license"
msgstr "Читать лицензию"

#: ../../client/solo/commands/_manage.py:725
msgid "Accept the license (I've read it)"
msgstr "Согласиться с лицензией (Я её прочитал(а))"

#: ../../client/solo/commands/_manage.py:730
msgid "Accept the license and don't ask anymore (I've read it)"
msgstr "Согласиться с лицензией и больше об этом не спрашивать (Я её прочитал(а))"

#: ../../client/solo/commands/_manage.py:734
msgid "Quit"
msgstr "Выйти"

#: ../../client/solo/commands/_manage.py:741
msgid "Your choice (type a number and press enter)"
msgstr "Ваш выбор (введите число и нажмите enter)"

#: ../../client/solo/commands/_manage.py:768
msgid "You need to accept the licenses below"
msgstr "Вам нужно согласиться с нижеуказанными лицензиями"

#: ../../client/solo/commands/_manage.py:776
msgid "needed by"
msgstr "необходимо для"

#: ../../client/solo/commands/_manage.py:816
msgid "No license data available"
msgstr ""

#: ../../client/solo/commands/_manage.py:827
msgid "No file viewer"
msgstr "Нет отобразителя файла"

#: ../../client/solo/commands/_manage.py:828
msgid "License saved into"
msgstr "Лицензия сохранена в"

#: ../../client/solo/commands/source.py:96
#: ../../client/solo/commands/install.py:116
#: ../../client/solo/commands/download.py:95
msgid "only include dependencies of selected packages"
msgstr ""

#: ../../client/solo/commands/source.py:115
#: ../../client/solo/commands/download.py:114
msgid "calculate dependencies relaxing constraints"
msgstr ""

#: ../../client/solo/commands/source.py:121
#: ../../client/solo/commands/install.py:104
#: ../../client/solo/commands/upgrade.py:99
#: ../../client/solo/commands/download.py:120
msgid "include build-time dependencies"
msgstr ""

#: ../../client/solo/commands/source.py:127
msgid "save files into the current working directory"
msgstr ""

#: ../../client/solo/commands/source.py:165
msgid "Source code download"
msgstr ""

#: ../../client/solo/commands/source.py:207
msgid "sources download"
msgstr ""

#: ../../client/solo/commands/source.py:229
msgid "download packages source code"
msgstr ""

#: ../../client/solo/commands/conf.py:75
msgid "manage configuration file updates"
msgstr ""

#: ../../client/solo/commands/conf.py:79
msgid "update configuration files"
msgstr ""

#: ../../client/solo/commands/conf.py:140
msgid "Scanning filesystem"
msgstr "Сканирование файловой системы"

#: ../../client/solo/commands/conf.py:146
#: ../../client/solo/commands/conf.py:172
msgid "All fine baby. Nothing to do!"
msgstr "Всё пучком. Нечего делать!"

#: ../../client/solo/commands/conf.py:167
msgid "file"
msgstr "файл"

#: ../../client/solo/commands/conf.py:183
#: ../../client/solo/commands/conf.py:629
msgid "Type a number"
msgstr ""

#: ../../client/solo/commands/conf.py:213
msgid "Please choose a file to update by typing its identification number."
msgstr "Выберите файл к обновлению, набрав его идентификационный номер"

#: ../../client/solo/commands/conf.py:217
msgid "Other options are:"
msgstr "Другие опции:"

#: ../../client/solo/commands/conf.py:221
#: ../../client/solo/commands/notice.py:153
#: ../../magneto/src/magneto/core/interfaces.py:191
msgid "Exit"
msgstr "Выход"

#: ../../client/solo/commands/conf.py:226
msgid "Automerge all the files asking you one by one"
msgstr "Автоматически объединить файлы, спрашивая Вас о каждом"

#: ../../client/solo/commands/conf.py:231
msgid "Automerge all the files without questioning"
msgstr "Автоматически объединить все файлы без вопросов"

#: ../../client/solo/commands/conf.py:236
msgid "Discard all the files asking you one by one"
msgstr "Пропустить все файлы, спрашивая Вас о каждом"

#: ../../client/solo/commands/conf.py:241
msgid "Discard all the files without questioning"
msgstr "Пропустить все файлы без вопросов"

#: ../../client/solo/commands/conf.py:254
msgid "Please choose an action to take for the selected file."
msgstr "Выберите, что сделать с этим файлом"

#: ../../client/solo/commands/conf.py:259
msgid "Come back to the files list"
msgstr "Обратно в список файлов"

#: ../../client/solo/commands/conf.py:264
msgid "Replace original with update"
msgstr "Заменить оригинал обновлением"

#: ../../client/solo/commands/conf.py:269
msgid "Delete update, keeping original as is"
msgstr "Удалить обновление и оставить оригинал, как было"

#: ../../client/solo/commands/conf.py:274
msgid "Edit proposed file and show diffs again"
msgstr "Редактировать предлагаемый файл и показать различия снова"

#: ../../client/solo/commands/conf.py:279
msgid "Interactively merge original with update"
msgstr ""

#: ../../client/solo/commands/conf.py:284
msgid "Show differences again"
msgstr "Показать различия снова"

#: ../../client/solo/commands/conf.py:380
#: ../../client/solo/commands/conf.py:435
msgid "Source file"
msgstr ""

#: ../../client/solo/commands/conf.py:385
#: ../../client/solo/commands/conf.py:440
msgid "Destination file"
msgstr ""

#: ../../client/solo/commands/conf.py:390
msgid "Overwrite ?"
msgstr "Переписать ?"

#: ../../client/solo/commands/conf.py:399
msgid "Cannot merge"
msgstr ""

#: ../../client/solo/commands/conf.py:445
msgid "Discard ?"
msgstr "Отклонить ?"

#: ../../client/solo/commands/conf.py:451
#: ../../client/solo/commands/conf.py:586
msgid "Discarding"
msgstr "Отклонено"

#: ../../client/solo/commands/conf.py:460
msgid "Cannot remove"
msgstr ""

#: ../../client/solo/commands/conf.py:476
msgid "Editing file"
msgstr "Редактирование файла"

#: ../../client/solo/commands/conf.py:483
msgid "Edited file"
msgstr "Редактированный файл"

#: ../../client/solo/commands/conf.py:485
msgid "showing difference"
msgstr ""

#: ../../client/solo/commands/conf.py:493
#: ../../client/solo/commands/conf.py:598
#: ../../client/solo/commands/conf.py:609
msgid "Automerging"
msgstr ""

#: ../../client/solo/commands/conf.py:534
msgid "Interactive merge"
msgstr ""

#: ../../client/solo/commands/conf.py:550
msgid "OSError during interactive merge"
msgstr ""

#: ../../client/solo/commands/conf.py:557
msgid "IOError during interactive merge"
msgstr ""

#: ../../client/solo/commands/conf.py:567
msgid "Unable to merge file"
msgstr ""

#: ../../client/solo/commands/conf.py:617
msgid "Selected file"
msgstr "Выбранный файл"

#: ../../client/solo/commands/conf.py:639
#, python-format
msgid "Replacing %s with %s"
msgstr ""

#: ../../client/solo/commands/conf.py:646
#, python-format
msgid "Cannot merge %s"
msgstr ""

#: ../../client/solo/commands/conf.py:654
#, python-format
msgid "Deleting %s"
msgstr ""

#: ../../client/solo/commands/conf.py:660
#, python-format
msgid "Cannot remove %s"
msgstr ""

#: ../../client/solo/commands/conf.py:698
msgid "manage package file updates"
msgstr ""

#: ../../client/solo/commands/match.py:82
msgid "match keyword"
msgstr ""

#: ../../client/solo/commands/match.py:94
msgid "return only injected packages "
msgstr ""

#: ../../client/solo/commands/match.py:99
msgid "match among installed packages only"
msgstr ""

#: ../../client/solo/commands/match.py:103
msgid "match among available packages only"
msgstr ""

#: ../../client/solo/commands/match.py:107
msgid "return all the matches, not just the best"
msgstr ""

#: ../../client/solo/commands/match.py:112
msgid "return matches found in every repository"
msgstr ""

#: ../../client/solo/commands/match.py:118
msgid "print repository information (w/--quiet)"
msgstr "отображает информацию репозитория (с --quiet)"

#: ../../client/solo/commands/match.py:122
msgid "print description too (w/--quiet)"
msgstr "показывает ещё и описание (с --quiet)"

#: ../../client/solo/commands/match.py:126
msgid "print \":<slot>\" next to package names (w/--quiet)"
msgstr ""

#: ../../client/solo/commands/match.py:208
msgid "Matching"
msgstr "Подсчёт"

#: ../../client/solo/commands/match.py:258
msgid "match packages in repositories"
msgstr ""

#: ../../client/solo/commands/install.py:147
#: ../../client/solo/commands/upgrade.py:142
msgid "relax dependencies constraints during calculation"
msgstr ""

#: ../../client/solo/commands/install.py:154
#: ../../client/solo/commands/upgrade.py:149
#: ../../client/solo/commands/download.py:127
msgid "download multiple packages in parallel (max 10)"
msgstr ""

#: ../../client/solo/commands/install.py:215
msgid "These are the packages that would be installed"
msgstr "Здесь перечислены пакеты, которые должны быть установлены"

#: ../../client/solo/commands/install.py:363
msgid "conflicting/substituted"
msgstr "конфликтуют/взаимозаменяют"

#: ../../client/solo/commands/install.py:398
msgid "Packages needing to be installed/updated/downgraded"
msgstr "Пакеты к установке/обновлению/обновлению до более старой версии"

#: ../../client/solo/commands/install.py:403
msgid "Packages needing to be removed"
msgstr "Пакеты к удалению"

#: ../../client/solo/commands/install.py:410
msgid "Packages needing to be installed"
msgstr "Пакеты к установке"

#: ../../client/solo/commands/install.py:417
msgid "Packages needing to be reinstalled"
msgstr "Пакеты к переустановке"

#: ../../client/solo/commands/install.py:424
msgid "Packages needing to be updated"
msgstr "Пакеты к обновлению"

#: ../../client/solo/commands/install.py:431
msgid "Packages needing to be downgraded"
msgstr "Пакеты, которые необходимо обновить до более старой версии"

#: ../../client/solo/commands/install.py:444
msgid "Download size"
msgstr "Размер загрузки"

#: ../../client/solo/commands/install.py:451
msgid "Used disk space"
msgstr "Использованное место на диске"

#: ../../client/solo/commands/install.py:467
msgid "You need at least"
msgstr "Вам нужно совсем немного"

#: ../../client/solo/commands/install.py:469
msgid "of free space"
msgstr "свободного места"

#: ../../client/solo/commands/install.py:483
msgid "You don't have enough space for the installation. Free some space into"
msgstr "У Вас недостаточно места для инсталляции. Освободите немного пространства."

#: ../../client/solo/commands/install.py:586
msgid "Download complete"
msgstr "Загрузка завершена"

#: ../../client/solo/commands/install.py:618
msgid "install"
msgstr "установить"

#: ../../client/solo/commands/install.py:640
msgid "Installation complete"
msgstr "Установка завершена"

#: ../../client/solo/commands/install.py:649
msgid "install or update packages or package files"
msgstr ""

#: ../../client/solo/commands/upgrade.py:129
msgid ""
"remove unmaintained packages, if any. This will respect --ask, --pretend and"
" other switches."
msgstr ""

#: ../../client/solo/commands/upgrade.py:198
msgid "Calculating System Updates"
msgstr "Подсчёт системных обновлений"

#: ../../client/solo/commands/upgrade.py:214
msgid "Packages matching update"
msgstr "Обновление пакетного подсчёта"

#: ../../client/solo/commands/upgrade.py:220
msgid "Packages matching not available"
msgstr "Пакетный подсчёт недоступен"

#: ../../client/solo/commands/upgrade.py:226
msgid "Packages matching already up to date"
msgstr "Пакетный подсчёт уже обновлён"

#: ../../client/solo/commands/upgrade.py:250
msgid "Nothing to update"
msgstr "Нечего обновлять"

#: ../../client/solo/commands/upgrade.py:264
msgid ""
"On the system there are packages that are not available anymore in the "
"online repositories"
msgstr "В системе имеются пакеты, которые более недоступны в онлайн-репозиториях"

#: ../../client/solo/commands/upgrade.py:269
msgid ""
"Even if they are usually harmless, it is suggested (after proper "
"verification) to remove them."
msgstr ""

#: ../../client/solo/commands/upgrade.py:281
msgid "To automatically remove them, please run equo with --purge."
msgstr ""

#: ../../client/solo/commands/upgrade.py:292
msgid "Selective"
msgstr ""

#: ../../client/solo/commands/upgrade.py:300
msgid "Would you like to remove them?"
msgstr ""

#: ../../client/solo/commands/upgrade.py:319
msgid "Remove this?"
msgstr ""

#: ../../client/solo/commands/upgrade.py:335
msgid "Nothing to remove"
msgstr "Нечего удалять"

#: ../../client/solo/commands/upgrade.py:363
msgid "There are more updates to install, reloading Entropy"
msgstr ""

#: ../../client/solo/commands/upgrade.py:382
msgid "upgrade the system"
msgstr ""

#: ../../client/solo/commands/rescue.py:91
msgid "check installed packages repository for errors"
msgstr ""

#: ../../client/solo/commands/rescue.py:98
msgid "compact the installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:104
msgid ""
"re-generate the installed packages repository using the Source Package "
"Manager"
msgstr ""

#: ../../client/solo/commands/rescue.py:111
msgid "re-generate SPM<->Entropy package UIDs mapping"
msgstr ""

#: ../../client/solo/commands/rescue.py:117
msgid ""
"update Entropy installed packages repository merging Source Package Manager "
"changes"
msgstr ""

#: ../../client/solo/commands/rescue.py:128
msgid "create a backup of the installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:134
msgid "restore a backup of the installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:173
msgid "Repository error"
msgstr ""

#: ../../client/solo/commands/rescue.py:181
#: ../../client/solo/commands/rescue.py:234
#: ../../client/solo/commands/rescue.py:243
msgid "Sanity Check"
msgstr "проверка работоспособности"

#: ../../client/solo/commands/rescue.py:182
msgid "installed packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:188
msgid "Scanning..."
msgstr "Сканирование..."

#: ../../client/solo/commands/rescue.py:224
msgid "Error checking package"
msgstr ""

#: ../../client/solo/commands/rescue.py:244
msgid "passed"
msgstr ""

#: ../../client/solo/commands/rescue.py:266
msgid "Compacting the Installed Packages repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:279
msgid "Compaction complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:308
msgid ""
"the Installed Packages repository will be re-generated using the Source "
"Package Manager"
msgstr ""

#: ../../client/solo/commands/rescue.py:319
msgid "I am not joking, this is quite disruptive"
msgstr ""

#: ../../client/solo/commands/rescue.py:327
msgid "Understood ?"
msgstr "Я ясно излагаю ?"

#: ../../client/solo/commands/rescue.py:331
msgid "Really ?"
msgstr "Правда ?"

#: ../../client/solo/commands/rescue.py:336
msgid "This is your last chance"
msgstr "Это твой последний шанс"

#: ../../client/solo/commands/rescue.py:337
msgid "Ok?"
msgstr "Ok?"

#: ../../client/solo/commands/rescue.py:354
#: ../../client/solo/commands/rescue.py:377
msgid "Cannot read metadata"
msgstr ""

#: ../../client/solo/commands/rescue.py:385
msgid "Creating a backup of the current repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:397
msgid "Cannot backup the repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:414
msgid "Cannot delete old repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:424
msgid "Initializing a new repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:445
msgid "Repository initialized, generating metadata"
msgstr ""

#: ../../client/solo/commands/rescue.py:475
#: ../../client/solo/commands/rescue.py:737
msgid "Invalid package"
msgstr ""

#: ../../client/solo/commands/rescue.py:488
#: ../../client/solo/commands/rescue.py:751
msgid "Metadata generation error"
msgstr ""

#: ../../client/solo/commands/rescue.py:531
msgid "Indexing metadata, please wait..."
msgstr ""

#: ../../client/solo/commands/rescue.py:536
msgid "Repository metadata generation complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:551
msgid "Scanning Source Package Manager repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:565
msgid "Cannot find package"
msgstr ""

#: ../../client/solo/commands/rescue.py:577
msgid "Scanning Entropy repository"
msgstr ""

#: ../../client/solo/commands/rescue.py:648
msgid "These packages were removed"
msgstr ""

#: ../../client/solo/commands/rescue.py:670
#: ../../client/solo/commands/rescue.py:710
msgid "Continue ?"
msgstr ""

#: ../../client/solo/commands/rescue.py:689
msgid "Removal complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:696
msgid "These packages were added"
msgstr ""

#: ../../client/solo/commands/rescue.py:793
msgid "Update complete"
msgstr ""

#: ../../client/solo/commands/rescue.py:805
msgid "Re-generating packages mapping"
msgstr ""

#: ../../client/solo/commands/rescue.py:814
msgid "Packages mapping re-generated"
msgstr ""

#: ../../client/solo/commands/rescue.py:842
msgid "No backups found"
msgstr ""

#: ../../client/solo/commands/rescue.py:864
msgid "Select the repository to restore"
msgstr ""

#: ../../client/solo/commands/rescue.py:871
msgid "Entropy Installed Packages Repository backups"
msgstr ""

#: ../../client/solo/commands/rescue.py:896
msgid "tools to rescue the running system"
msgstr ""

#: ../../client/solo/commands/cleanup.py:97
msgid "remove downloaded packages and clean temp. directories"
msgstr "удалить загруженные пакеты и очистить временные директории"

#: ../../client/solo/commands/hop.py:92
msgid "Already on branch"
msgstr "Уже на ветке"

#: ../../client/solo/commands/hop.py:133
msgid "Succesfully switched to branch"
msgstr "Успешно переключено на ветку"

#: ../../client/solo/commands/hop.py:140
msgid "Now run 'equo upgrade' to upgrade your distribution to"
msgstr ""

#: ../../client/solo/commands/hop.py:148
msgid "Unable to switch to branch"
msgstr "Невозможно переключить ветку"

#: ../../client/solo/commands/hop.py:160
msgid "upgrade the System to a new branch"
msgstr ""

#: ../../client/solo/commands/download.py:165
msgid "Package download"
msgstr ""

#: ../../client/solo/commands/download.py:202
msgid "download packages, essentially"
msgstr ""

#: ../../client/solo/commands/version.py:59
msgid "show equo version"
msgstr ""

#: ../../client/solo/commands/mask.py:53
msgid "only show what would be done"
msgstr "показывать только то, что будет сделано"

#: ../../client/solo/commands/mask.py:125
msgid "These are the packages that would be handled"
msgstr ""

#: ../../client/solo/commands/mask.py:160
msgid "Would you like to continue?"
msgstr ""

#: ../../client/solo/commands/mask.py:174
msgid "action not executed"
msgstr ""

#: ../../client/solo/commands/mask.py:222
msgid "mask one or more packages"
msgstr ""

#: ../../client/solo/commands/mask.py:229
msgid "unmask one or more packages"
msgstr ""

#: ../../client/solo/commands/notice.py:162
msgid "Choose one by typing its identifier"
msgstr "Выберите один, набрав его название"

#: ../../client/solo/commands/notice.py:194
msgid "Link"
msgstr "Ссылка"

#: ../../client/solo/commands/notice.py:203
msgid "Press Enter to continue"
msgstr "Нажмите Enter, чтобы продолжить"

#: ../../client/solo/commands/notice.py:211
msgid "repository notice board reader"
msgstr "просмотрщик доски уведомлений репозитория"

#: ../../magneto/src/magneto/kde/interfaces.py:74
msgid "Magneto Entropy Updates Applet"
msgstr "Апплет обновлений Entropy Magneto"

#: ../../magneto/src/magneto/kde/interfaces.py:224
#: ../../magneto/src/magneto/gtk/interfaces.py:210
msgid "Your system currently has an outdated version of"
msgstr "На Вашей системе в данный момент имеется устаревшая версия"

#: ../../magneto/src/magneto/kde/interfaces.py:225
#: ../../magneto/src/magneto/gtk/interfaces.py:211
msgid "installed"
msgstr "установлено"

#: ../../magneto/src/magneto/kde/interfaces.py:226
#: ../../magneto/src/magneto/gtk/interfaces.py:212
msgid "the latest available version is"
msgstr "наиболее актуальная версия -"

#: ../../magneto/src/magneto/kde/interfaces.py:228
#: ../../magneto/src/magneto/gtk/interfaces.py:214
msgid ""
"It is recommended that you upgrade to the latest before updating any other "
"packages"
msgstr "Рекомендуется обновить до последней версии перед обновлением каких-либо других пакетов"

#: ../../magneto/src/magneto/kde/components.py:48
msgid "Close"
msgstr "Закрыть"

#: ../../magneto/src/magneto/kde/components.py:49
msgid "Launch Application Browser"
msgstr ""

#: ../../magneto/src/magneto/kde/components.py:59
msgid "Application updates"
msgstr "Обновления приложений"

#: ../../magneto/src/magneto/gtk/interfaces.py:146
msgid "System Updates"
msgstr ""

#: ../../magneto/src/magneto/gtk/components.py:52
msgid "Application"
msgstr "Приложения"

#: ../../magneto/src/magneto/gtk/components.py:54
msgid "Latest version"
msgstr "Последняя версия"

#: ../../magneto/src/magneto/gtk/components.py:96
msgid "Critical Information"
msgstr "Критическая информация"

#: ../../magneto/src/magneto/core/interfaces.py:178
msgid "_Disable Notification Applet"
msgstr "_Деактивировать апплет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:179
msgid "Disable Notification Applet"
msgstr "_Деактивировать апплет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:180
msgid "_Enable Notification Applet"
msgstr "_Aктивировать апплет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:181
msgid "Enable Notification Applet"
msgstr "Активировать пакет уведомлений"

#: ../../magneto/src/magneto/core/interfaces.py:182
msgid "_Check for updates"
msgstr "_Проверить на обновления"

#: ../../magneto/src/magneto/core/interfaces.py:183
msgid "Check for updates"
msgstr "Проверить на обновления"

#: ../../magneto/src/magneto/core/interfaces.py:184
msgid "_Launch Package Manager"
msgstr "_Запустить пакетный менеджер"

#: ../../magneto/src/magneto/core/interfaces.py:185
msgid "Launch Package Manager"
msgstr "Запустить пакетный менеджер"

#: ../../magneto/src/magneto/core/interfaces.py:186
msgid "_Packages Website"
msgstr "_Веб-страница пакетов"

#: ../../magneto/src/magneto/core/interfaces.py:187
msgid "Use Packages web interface"
msgstr "Использовать веб-интерфейс пакетов"

#: ../../magneto/src/magneto/core/interfaces.py:188
msgid "_Sabayon Linux Website"
msgstr "Веб-_сайт Sabayon Linux"

#: ../../magneto/src/magneto/core/interfaces.py:189
msgid "Launch Sabayon Linux Website"
msgstr "Открыть сайт Sabayon Linux"

#: ../../magneto/src/magneto/core/interfaces.py:191
msgid "_Exit"
msgstr "_Выход"

#: ../../magneto/src/magneto/core/interfaces.py:323
msgid "Cannot monitor Sabayon updates"
msgstr "Невозможно отобразить обновления Sabayon"

#: ../../magneto/src/magneto/core/interfaces.py:325
msgid "Entropy DBus service not available"
msgstr "Служба DBus Entropy недоступна"

#: ../../magneto/src/magneto/core/interfaces.py:326
msgid "unable to communicate with the updates service"
msgstr "невозможно соединиться со службой обновлений"

#: ../../magneto/src/magneto/core/interfaces.py:341
#, python-format
msgid "There is %s update available"
msgid_plural "There are %s updates available"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#: ../../magneto/src/magneto/core/interfaces.py:347
msgid "Sabayon updates available"
msgstr "Обновления Sabayon доступны"

#: ../../magneto/src/magneto/core/interfaces.py:348
#, python-format
msgid "There is <b>%s</b> update available"
msgid_plural "There are <b>%s</b> updates available"
msgstr[0] ""
msgstr[1] ""
msgstr[2] ""

#. all fine, no updates
#: ../../magneto/src/magneto/core/interfaces.py:356
#: ../../magneto/src/magneto/core/interfaces.py:358
msgid "Your Sabayon is up-to-date"
msgstr "Ваш Sabayon уже обновлен до последней версии"

#: ../../magneto/src/magneto/core/interfaces.py:359
msgid "No updates available at this time, cool!"
msgstr "На текущий момент обновлений не обнаружено, отлично!"

#. all fine, no updates
#: ../../magneto/src/magneto/core/interfaces.py:370
msgid "Repositories are being updated"
msgstr "Репозитории обновляются"

#: ../../magneto/src/magneto/core/interfaces.py:371
msgid "Sabayon repositories status"
msgstr "Статус репозиториев Sabayon"

#: ../../magneto/src/magneto/core/interfaces.py:372
msgid "Repositories are being updated automatically"
msgstr "Репозитории обновляются автоматически"

#: ../../magneto/src/magneto/core/interfaces.py:462
msgid "Updates Notification Applet Disabled"
msgstr "Апплет уведомления об обновлениях отключён"

#: ../../magneto/src/magneto/core/interfaces.py:472
msgid "Updates Notification Applet Enabled"
msgstr "Апплет уведомления об обновлениях включён"

#: ../kernel-switcher:27
msgid "Sabayon Kernel Switcher"
msgstr ""

#: ../kernel-switcher:34
msgid "quiet mode"
msgstr ""

#: ../kernel-switcher:37
msgid "verbose mode"
msgstr ""

#: ../kernel-switcher:41
msgid "ask confirmation"
msgstr ""

#: ../kernel-switcher:89
msgid "Package does not exist"
msgstr ""

#: ../kernel-switcher:97
msgid "Not a kernel"
msgstr ""

#: ../kernel-switcher:107
msgid "Cannot find your currently running kernel."
msgstr ""

#: ../kernel-switcher:109
msgid "Try without --from-running."
msgstr ""

#: ../kernel-switcher:124
msgid "No kernel packages found"
msgstr ""

#: ../kernel-switcher:138
msgid "install a new or just another kernel"
msgstr ""

#: ../kernel-switcher:143
msgid "use 'uname -r' to determine the running kernel"
msgstr ""

#: ../kernel-switcher:146
msgid "the new kernel package dependency name"
msgstr ""

#: ../kernel-switcher:150
msgid "list kernels"
msgstr ""

#: ../../magneto/src/magneto/gtk/magneto.glade.h:1
msgid "Magneto Application Updates Notifier"
msgstr "Уведомитель об обновлениях приложений Magneto"

#: ../../magneto/src/magneto/gtk/magneto.glade.h:3
msgid "_Load Package Manager"
msgstr "_Загрузить пакетный менеджер"
